<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List href="AMeeGuS2022_BookOfAbstracts_20220128.fld/filelist.xml">
<link rel=Edit-Time-Data
href="AMeeGuS2022_BookOfAbstracts_20220128.fld/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Romana Yanez</o:Author>
  <o:LastAuthor>Romana Yanez</o:LastAuthor>
  <o:Revision>2</o:Revision>
  <o:TotalTime>1</o:TotalTime>
  <o:LastPrinted>2022-01-21T16:04:00Z</o:LastPrinted>
  <o:Created>2022-01-28T19:04:00Z</o:Created>
  <o:LastSaved>2022-01-28T19:04:00Z</o:LastSaved>
  <o:Pages>35</o:Pages>
  <o:Words>13077</o:Words>
  <o:Characters>74539</o:Characters>
  <o:Lines>621</o:Lines>
  <o:Paragraphs>174</o:Paragraphs>
  <o:CharactersWithSpaces>87442</o:CharactersWithSpaces>
  <o:Version>16.00</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:RelyOnVML/>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=dataStoreItem
href="AMeeGuS2022_BookOfAbstracts_20220128.fld/item0001.xml"
target="AMeeGuS2022_BookOfAbstracts_20220128.fld/props002.xml">
<link rel=themeData
href="AMeeGuS2022_BookOfAbstracts_20220128.fld/themedata.thmx">
<link rel=colorSchemeMapping
href="AMeeGuS2022_BookOfAbstracts_20220128.fld/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:SpellingState>Clean</w:SpellingState>
  <w:GrammarState>Clean</w:GrammarState>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>en-PT</w:LidThemeOther>
  <w:LidThemeAsian>X-NONE</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="false"
  DefSemiHidden="false" DefQFormat="false" DefPriority="99"
  LatentStyleCount="376">
  <w:LsdException Locked="false" Priority="0" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 9"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 9"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footnote text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="header"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footer"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index heading"/>
  <w:LsdException Locked="false" Priority="35" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="table of figures"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="envelope address"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="envelope return"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footnote reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="line number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="page number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="endnote reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="endnote text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="table of authorities"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="macro"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="toa heading"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 5"/>
  <w:LsdException Locked="false" Priority="10" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Closing"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Signature"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="true"
   UnhideWhenUsed="true" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Message Header"/>
  <w:LsdException Locked="false" Priority="11" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Salutation"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Date"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text First Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text First Indent 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Note Heading"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Block Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Hyperlink"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="FollowedHyperlink"/>
  <w:LsdException Locked="false" Priority="22" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Document Map"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Plain Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="E-mail Signature"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Top of Form"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Bottom of Form"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal (Web)"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Acronym"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Address"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Cite"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Code"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Definition"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Keyboard"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Preformatted"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Sample"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Typewriter"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Variable"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal Table"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation subject"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="No List"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Contemporary"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Elegant"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Professional"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Subtle 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Subtle 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Balloon Text"/>
  <w:LsdException Locked="false" Priority="39" Name="Table Grid"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Theme"/>
  <w:LsdException Locked="false" SemiHidden="true" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" SemiHidden="true" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" QFormat="true"
   Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" QFormat="true"
   Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" QFormat="true"
   Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" QFormat="true"
   Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" QFormat="true"
   Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" QFormat="true"
   Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" SemiHidden="true"
   UnhideWhenUsed="true" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="TOC Heading"/>
  <w:LsdException Locked="false" Priority="41" Name="Plain Table 1"/>
  <w:LsdException Locked="false" Priority="42" Name="Plain Table 2"/>
  <w:LsdException Locked="false" Priority="43" Name="Plain Table 3"/>
  <w:LsdException Locked="false" Priority="44" Name="Plain Table 4"/>
  <w:LsdException Locked="false" Priority="45" Name="Plain Table 5"/>
  <w:LsdException Locked="false" Priority="40" Name="Grid Table Light"/>
  <w:LsdException Locked="false" Priority="46" Name="Grid Table 1 Light"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark"/>
  <w:LsdException Locked="false" Priority="51" Name="Grid Table 6 Colorful"/>
  <w:LsdException Locked="false" Priority="52" Name="Grid Table 7 Colorful"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 1"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 1"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 1"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 2"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 2"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 2"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 3"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 3"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 3"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 4"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 4"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 4"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 5"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 5"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 5"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 6"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 6"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 6"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="46" Name="List Table 1 Light"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark"/>
  <w:LsdException Locked="false" Priority="51" Name="List Table 6 Colorful"/>
  <w:LsdException Locked="false" Priority="52" Name="List Table 7 Colorful"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 1"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 1"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 1"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 2"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 2"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 2"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 3"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 3"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 3"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 4"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 4"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 4"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 5"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 5"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 5"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 6"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 6"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 6"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Mention"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Smart Hyperlink"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Hashtag"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Unresolved Mention"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Smart Link"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:"MS Gothic";
	panose-1:2 11 6 9 7 2 5 8 2 4;
	mso-font-alt:"ＭＳ ゴシック";
	mso-font-charset:128;
	mso-generic-font-family:modern;
	mso-font-pitch:fixed;
	mso-font-signature:-536870145 1791491579 134217746 0 131231 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-pitch:variable;
	mso-font-signature:-536870145 1107305727 0 0 415 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073732485 9 0 511 0;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073732485 9 0 511 0;}
@font-face
	{font-family:"Tw Cen MT";
	panose-1:2 11 6 2 2 1 4 2 6 3;
	mso-font-charset:77;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:3 0 0 0 3 0;}
@font-face
	{font-family:"Calibri Light \(Headings\)";
	panose-1:2 11 6 4 2 2 2 2 2 4;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-pitch:auto;
	mso-font-signature:0 0 0 0 0 0;}
@font-face
	{font-family:"\@MS Gothic";
	panose-1:2 11 6 9 7 2 5 8 2 4;
	mso-font-charset:128;
	mso-generic-font-family:modern;
	mso-font-pitch:fixed;
	mso-font-signature:-536870145 1791491579 134217746 0 131231 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	mso-style-next:Normal1;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	color:black;
	mso-themecolor:text1;}
h1
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 1 Char";
	mso-style-next:Normal;
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:center;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	background:#22A09C;
	font-size:18.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:white;
	mso-themecolor:background1;
	mso-font-kerning:0pt;
	mso-ansi-language:EN-GB;}
h2
	{mso-style-priority:9;
	mso-style-qformat:yes;
	mso-style-link:"Heading 2 Char";
	mso-style-next:Normal;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:2;
	font-size:14.0pt;
	mso-bidi-font-size:13.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Calibri Light \(Headings\)";
	color:#5C95F3;
	mso-ansi-language:EN-GB;
	font-weight:normal;}
h3
	{mso-style-priority:9;
	mso-style-qformat:yes;
	mso-style-link:"Heading 3 Char";
	mso-style-next:Normal;
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:3;
	font-size:12.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:red;
	font-weight:normal;}
h4
	{mso-style-priority:9;
	mso-style-qformat:yes;
	mso-style-link:"Heading 4 Char";
	mso-style-next:Normal;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:4;
	font-size:10.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#70AD47;
	mso-themecolor:accent6;
	mso-ansi-language:PT;
	mso-bidi-font-weight:normal;
	font-style:italic;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;
	font-weight:bold;
	font-style:italic;}
p.MsoToc2, li.MsoToc2, div.MsoToc2
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:12.0pt;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;
	font-weight:bold;}
p.MsoToc3, li.MsoToc3, div.MsoToc3
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:24.0pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;}
p.MsoToc4, li.MsoToc4, div.MsoToc4
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;}
p.MsoToc5, li.MsoToc5, div.MsoToc5
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:48.0pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;}
p.MsoToc6, li.MsoToc6, div.MsoToc6
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:60.0pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;}
p.MsoToc7, li.MsoToc7, div.MsoToc7
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:72.0pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;}
p.MsoToc8, li.MsoToc8, div.MsoToc8
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:84.0pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;}
p.MsoToc9, li.MsoToc9, div.MsoToc9
	{mso-style-update:auto;
	mso-style-priority:39;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:96.0pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-priority:99;
	mso-style-link:"Header Char";
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan;
	tab-stops:center 225.65pt right 451.3pt;
	font-size:12.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	color:black;
	mso-themecolor:text1;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-priority:99;
	mso-style-link:"Footer Char";
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan;
	tab-stops:center 225.65pt right 451.3pt;
	font-size:12.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	color:black;
	mso-themecolor:text1;}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{mso-style-update:auto;
	mso-style-priority:10;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Title Char";
	mso-style-next:"Heading 2";
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:16.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	color:black;
	mso-themecolor:text1;}
p.MsoSubtitle, li.MsoSubtitle, div.MsoSubtitle
	{mso-style-priority:11;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Subtitle Char";
	mso-style-next:Normal;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	mso-add-space:auto;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:#5A5A5A;
	mso-themecolor:text1;
	mso-themetint:165;
	letter-spacing:.75pt;}
p.MsoSubtitleCxSpFirst, li.MsoSubtitleCxSpFirst, div.MsoSubtitleCxSpFirst
	{mso-style-priority:11;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Subtitle Char";
	mso-style-next:Normal;
	mso-style-type:export-only;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	mso-add-space:auto;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:#5A5A5A;
	mso-themecolor:text1;
	mso-themetint:165;
	letter-spacing:.75pt;}
p.MsoSubtitleCxSpMiddle, li.MsoSubtitleCxSpMiddle, div.MsoSubtitleCxSpMiddle
	{mso-style-priority:11;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Subtitle Char";
	mso-style-next:Normal;
	mso-style-type:export-only;
	margin:0cm;
	mso-add-space:auto;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:#5A5A5A;
	mso-themecolor:text1;
	mso-themetint:165;
	letter-spacing:.75pt;}
p.MsoSubtitleCxSpLast, li.MsoSubtitleCxSpLast, div.MsoSubtitleCxSpLast
	{mso-style-priority:11;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Subtitle Char";
	mso-style-next:Normal;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	mso-add-space:auto;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:#5A5A5A;
	mso-themecolor:text1;
	mso-themetint:165;
	letter-spacing:.75pt;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:#0563C1;
	mso-themecolor:hyperlink;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:#954F72;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
span.MsoSubtleReference
	{mso-style-priority:31;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	font-variant:small-caps;
	color:#5A5A5A;
	mso-themecolor:text1;
	mso-themetint:165;}
p.MsoTocHeading, li.MsoTocHeading, div.MsoTocHeading
	{mso-style-priority:39;
	mso-style-qformat:yes;
	mso-style-parent:"Heading 1";
	mso-style-next:Normal;
	margin-top:24.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	line-height:115%;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#2F5496;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-ansi-language:EN-US;
	mso-fareast-language:EN-US;
	font-weight:bold;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 1";
	mso-ansi-font-size:18.0pt;
	mso-bidi-font-size:18.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:white;
	mso-themecolor:background1;
	background:#22A09C;
	mso-ansi-language:EN-GB;
	mso-fareast-language:EN-GB;
	font-weight:bold;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 2";
	mso-ansi-font-size:14.0pt;
	mso-bidi-font-size:13.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Calibri Light \(Headings\)";
	color:#5C95F3;
	mso-ansi-language:EN-GB;
	mso-fareast-language:EN-GB;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 3";
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:red;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 4";
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#70AD47;
	mso-themecolor:accent6;
	mso-ansi-language:PT;
	mso-fareast-language:EN-GB;
	font-weight:bold;
	mso-bidi-font-weight:normal;
	font-style:italic;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Header;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Footer;}
span.TitleChar
	{mso-style-name:"Title Char";
	mso-style-priority:10;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Title;
	mso-ansi-font-size:16.0pt;
	mso-bidi-font-size:16.0pt;
	font-family:"Times New Roman",serif;
	mso-ascii-font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:"Times New Roman";
	mso-bidi-font-family:"Times New Roman";
	color:black;
	mso-themecolor:text1;
	mso-fareast-language:EN-GB;}
span.SubtitleChar
	{mso-style-name:"Subtitle Char";
	mso-style-priority:11;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Subtitle;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	font-family:"Times New Roman",serif;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:minor-fareast;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:#5A5A5A;
	mso-themecolor:text1;
	mso-themetint:165;
	letter-spacing:.75pt;
	mso-ansi-language:EN-GB;
	mso-fareast-language:EN-GB;}
p.Normal1, li.Normal1, div.Normal1
	{mso-style-name:Normal1;
	mso-style-unhide:no;
	mso-style-next:Normal;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:Calibri;
	mso-bidi-theme-font:minor-latin;
	color:black;
	mso-themecolor:text1;}
span.il
	{mso-style-name:il;
	mso-style-unhide:no;}
p.RYHEADING2, li.RYHEADING2, div.RYHEADING2
	{mso-style-name:RY_HEADING2;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	text-align:justify;
	mso-pagination:widow-orphan;
	font-size:16.0pt;
	mso-bidi-font-size:12.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Calibri Light";
	mso-bidi-theme-font:major-latin;
	color:black;
	mso-themecolor:text1;
	font-style:italic;}
span.SpellE
	{mso-style-name:"";
	mso-spl-e:yes;}
span.GramE
	{mso-style-name:"";
	mso-gram-e:yes;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
.MsoPapDefault
	{mso-style-type:export-only;
	margin-top:6.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;}
 /* Page Definitions */
 @page
	{mso-footnote-separator:url("AMeeGuS2022_BookOfAbstracts_20220128.fld/header.html") fs;
	mso-footnote-continuation-separator:url("AMeeGuS2022_BookOfAbstracts_20220128.fld/header.html") fcs;
	mso-endnote-separator:url("AMeeGuS2022_BookOfAbstracts_20220128.fld/header.html") es;
	mso-endnote-continuation-separator:url("AMeeGuS2022_BookOfAbstracts_20220128.fld/header.html") ecs;}
@page WordSection1
	{size:595.3pt 841.9pt;
	margin:72.0pt 72.0pt 72.0pt 72.0pt;
	mso-header-margin:20.65pt;
	mso-footer-margin:35.4pt;
	border:solid #22A09C 6.0pt;
	padding:24.0pt 24.0pt 24.0pt 24.0pt;
	mso-page-border-display:first-page;
	mso-title-page:yes;
	mso-header:url("AMeeGuS2022_BookOfAbstracts_20220128.fld/header.html") h1;
	mso-even-footer:url("AMeeGuS2022_BookOfAbstracts_20220128.fld/header.html") ef1;
	mso-footer:url("AMeeGuS2022_BookOfAbstracts_20220128.fld/header.html") f1;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 @list l0
	{mso-list-id:-132;
	mso-list-type:simple;
	mso-list-template-ids:323645142;}
@list l0:level1
	{mso-level-tab-stop:74.6pt;
	mso-level-number-position:left;
	margin-left:74.6pt;
	text-indent:-18.0pt;}
@list l1
	{mso-list-id:-131;
	mso-list-type:simple;
	mso-list-template-ids:398257598;}
@list l1:level1
	{mso-level-tab-stop:60.45pt;
	mso-level-number-position:left;
	margin-left:60.45pt;
	text-indent:-18.0pt;}
@list l2
	{mso-list-id:-130;
	mso-list-type:simple;
	mso-list-template-ids:482909370;}
@list l2:level1
	{mso-level-tab-stop:46.3pt;
	mso-level-number-position:left;
	margin-left:46.3pt;
	text-indent:-18.0pt;}
@list l3
	{mso-list-id:-129;
	mso-list-type:simple;
	mso-list-template-ids:2126513854;}
@list l3:level1
	{mso-level-tab-stop:32.15pt;
	mso-level-number-position:left;
	margin-left:32.15pt;
	text-indent:-18.0pt;}
@list l4
	{mso-list-id:-128;
	mso-list-type:simple;
	mso-list-template-ids:-828737756;}
@list l4:level1
	{mso-level-number-format:bullet;
	mso-level-text:;
	mso-level-tab-stop:74.6pt;
	mso-level-number-position:left;
	margin-left:74.6pt;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l5
	{mso-list-id:-127;
	mso-list-type:simple;
	mso-list-template-ids:858713232;}
@list l5:level1
	{mso-level-number-format:bullet;
	mso-level-text:;
	mso-level-tab-stop:60.45pt;
	mso-level-number-position:left;
	margin-left:60.45pt;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l6
	{mso-list-id:-126;
	mso-list-type:simple;
	mso-list-template-ids:-1266914690;}
@list l6:level1
	{mso-level-number-format:bullet;
	mso-level-text:;
	mso-level-tab-stop:46.3pt;
	mso-level-number-position:left;
	margin-left:46.3pt;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l7
	{mso-list-id:-125;
	mso-list-type:simple;
	mso-list-template-ids:-606424154;}
@list l7:level1
	{mso-level-number-format:bullet;
	mso-level-text:;
	mso-level-tab-stop:32.15pt;
	mso-level-number-position:left;
	margin-left:32.15pt;
	text-indent:-18.0pt;
	font-family:Symbol;}
@list l8
	{mso-list-id:-120;
	mso-list-type:simple;
	mso-list-template-ids:1333277336;}
@list l8:level1
	{mso-level-tab-stop:18.0pt;
	mso-level-number-position:left;
	margin-left:18.0pt;
	text-indent:-18.0pt;}
@list l9
	{mso-list-id:-119;
	mso-list-type:simple;
	mso-list-template-ids:1987602450;}
@list l9:level1
	{mso-level-number-format:bullet;
	mso-level-text:;
	mso-level-tab-stop:18.0pt;
	mso-level-number-position:left;
	margin-left:18.0pt;
	text-indent:-18.0pt;
	font-family:Symbol;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin-top:6.0pt;
	mso-para-margin-right:0cm;
	mso-para-margin-bottom:6.0pt;
	mso-para-margin-left:0cm;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1027"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=en-PT link="#0563C1" vlink="#954F72" style='tab-interval:36.0pt;
word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal align=center style='text-align:center'><v:shapetype id="_x0000_t75"
 coordsize="21600,21600" o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe"
 filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Picture_x0020_74" o:spid="_x0000_s1026" type="#_x0000_t75"
 style='position:absolute;left:0;text-align:left;margin-left:-24.55pt;
 margin-top:50.55pt;width:7in;height:739.8pt;z-index:251658240;visibility:visible;
 mso-wrap-style:square;mso-width-percent:0;mso-height-percent:0;
 mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:absolute;
 mso-position-horizontal-relative:text;mso-position-vertical:absolute;
 mso-position-vertical-relative:page;mso-width-percent:0;mso-height-percent:0;
 mso-width-relative:page;mso-height-relative:page'>
 <v:imagedata src="AMeeGuS2022_BookOfAbstracts_20220128.fld/image001.emz"
  o:title=""/>
 <w:wrap type="square" anchory="page"/>
</v:shape><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><span lang=EN-GB
style='mso-ansi-language:EN-GB;mso-no-proof:yes'><v:shape id="Picture_x0020_75"
 o:spid="_x0000_i1032" type="#_x0000_t75" alt="Graphical user interface, text, application&#10;&#10;Description automatically generated"
 style='width:415pt;height:248pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="AMeeGuS2022_BookOfAbstracts_20220128.fld/image002.png"
  o:title="Graphical user interface, text, application&#10;&#10;Description automatically generated"
  croptop="2973f" cropbottom="14390f" cropleft="3336f" cropright="1907f"/>
</v:shape></span><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=Normal1 align=center style='text-align:center'><span lang=EN-GB
style='mso-ansi-language:EN-GB;mso-no-proof:yes'><v:shape id="Picture_x0020_76"
 o:spid="_x0000_i1031" type="#_x0000_t75" alt="A picture containing graphical user interface&#10;&#10;Description automatically generated"
 style='width:451pt;height:275pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="AMeeGuS2022_BookOfAbstracts_20220128.fld/image003.png"
  o:title="A picture containing graphical user interface&#10;&#10;Description automatically generated"
  croptop="8565f"/>
</v:shape></span><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<w:Sdt SdtDocPart="t" DocPartType="Table of Contents" DocPartUnique="t"
 ID="-1928259990">
 <p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p><w:sdtPr></w:sdtPr></span></p>
 <p class=MsoNormal align=center style='text-align:center;background:#22A09C'><b><span
 lang=EN-GB style='font-size:18.0pt;font-family:"Calibri Light",sans-serif;
 mso-ascii-theme-font:major-latin;mso-hansi-theme-font:major-latin;mso-bidi-theme-font:
 major-latin;color:white;mso-themecolor:background1;mso-ansi-language:EN-GB'>Table
 of Contents<o:p></o:p></span></b></p>
 <p class=MsoToc1 style='tab-stops:right 450.8pt'><!--[if supportFields]><u><span
 lang=EN-GB style='font-size:11.0pt;text-transform:uppercase;mso-ansi-language:
 EN-GB;font-weight:normal;font-style:normal'><span style='mso-element:field-begin'></span><span
 style='mso-spacerun:yes'> </span>TOC \o &quot;1-1&quot; \h \z \u <span
 style='mso-element:field-separator'></span></span></u><![endif]--><span
 class=MsoHyperlink><span style='mso-no-proof:yes'><a href="#_Toc94288968">Keynote
 Lecturers<span style='color:black;mso-themecolor:text1;display:none;
 mso-hide:screen;text-decoration:none;text-underline:none'><span
 style='mso-tab-count:1'>  </span></span><!--[if supportFields]><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-begin'></span></span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'> PAGEREF _Toc94288968 \h </span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-separator'></span></span><![endif]--><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'>3</span><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><!--[if gte mso 9]><xml>
  <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000D0000005F0054006F006300390034003200380038003900360038000000</w:data>
 </xml><![endif]--></span><!--[if supportFields]><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><span style='mso-element:field-end'></span></span><![endif]--></a></span></span><span
 style='mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:minor-fareast;
 mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
 color:windowtext;font-weight:normal;font-style:normal;mso-no-proof:yes'><o:p></o:p></span></p>
 <p class=MsoToc1 style='tab-stops:right 450.8pt'><span class=MsoHyperlink><span
 style='mso-no-proof:yes'><a href="#_Toc94288969">Alumni<span style='color:
 black;mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><span style='mso-tab-count:1'> </span></span><!--[if supportFields]><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-begin'></span></span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'> PAGEREF _Toc94288969 \h </span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-separator'></span></span><![endif]--><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'>5</span><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><!--[if gte mso 9]><xml>
  <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000D0000005F0054006F006300390034003200380038003900360039000000</w:data>
 </xml><![endif]--></span><!--[if supportFields]><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><span style='mso-element:field-end'></span></span><![endif]--></a></span></span><span
 style='mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:minor-fareast;
 mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
 color:windowtext;font-weight:normal;font-style:normal;mso-no-proof:yes'><o:p></o:p></span></p>
 <p class=MsoToc1 style='tab-stops:right 450.8pt'><span class=MsoHyperlink><span
 style='mso-no-proof:yes'><a href="#_Toc94288970">Long Talks<span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-tab-count:1'>  </span></span><!--[if supportFields]><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-begin'></span></span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'> PAGEREF _Toc94288970 \h </span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-separator'></span></span><![endif]--><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'>7</span><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><!--[if gte mso 9]><xml>
  <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000D0000005F0054006F006300390034003200380038003900370030000000</w:data>
 </xml><![endif]--></span><!--[if supportFields]><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><span style='mso-element:field-end'></span></span><![endif]--></a></span></span><span
 style='mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:minor-fareast;
 mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
 color:windowtext;font-weight:normal;font-style:normal;mso-no-proof:yes'><o:p></o:p></span></p>
 <p class=MsoToc1 style='tab-stops:right 450.8pt'><span class=MsoHyperlink><span
 style='mso-no-proof:yes'><a href="#_Toc94288971">Short Talks<span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-tab-count:1'>  </span></span><!--[if supportFields]><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-begin'></span></span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'> PAGEREF _Toc94288971 \h </span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-separator'></span></span><![endif]--><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'>16</span><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><!--[if gte mso 9]><xml>
  <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000D0000005F0054006F006300390034003200380038003900370031000000</w:data>
 </xml><![endif]--></span><!--[if supportFields]><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><span style='mso-element:field-end'></span></span><![endif]--></a></span></span><span
 style='mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:minor-fareast;
 mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
 color:windowtext;font-weight:normal;font-style:normal;mso-no-proof:yes'><o:p></o:p></span></p>
 <p class=MsoToc1 style='tab-stops:right 450.8pt'><span class=MsoHyperlink><span
 style='mso-no-proof:yes'><a href="#_Toc94288972">Posters<span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-tab-count:1'>  </span></span><!--[if supportFields]><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-begin'></span></span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'> PAGEREF _Toc94288972 \h </span><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'><span style='mso-element:field-separator'></span></span><![endif]--><span
 style='color:black;mso-themecolor:text1;display:none;mso-hide:screen;
 text-decoration:none;text-underline:none'>20</span><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><!--[if gte mso 9]><xml>
  <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000D0000005F0054006F006300390034003200380038003900370032000000</w:data>
 </xml><![endif]--></span><!--[if supportFields]><span style='color:black;
 mso-themecolor:text1;display:none;mso-hide:screen;text-decoration:none;
 text-underline:none'><span style='mso-element:field-end'></span></span><![endif]--></a></span></span><span
 style='mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:minor-fareast;
 mso-bidi-font-family:"Times New Roman";mso-bidi-theme-font:minor-bidi;
 color:windowtext;font-weight:normal;font-style:normal;mso-no-proof:yes'><o:p></o:p></span></p>
 <p class=MsoNormal><!--[if supportFields]><b><i><u><span lang=EN-GB
 style='font-size:11.0pt;mso-bidi-font-family:Calibri;mso-bidi-theme-font:minor-latin;
 text-transform:uppercase;mso-ansi-language:EN-GB'><span style='mso-element:
 field-end'></span></span></u></i></b><![endif]--><b><i><u><span lang=EN-GB
 style='font-size:11.0pt;mso-bidi-font-family:Calibri;mso-bidi-theme-font:minor-latin;
 text-transform:uppercase;mso-ansi-language:EN-GB'><o:p><span style='text-decoration:
  none'>&nbsp;</span></o:p></span></u></i></b></p>
</w:Sdt>

<h1><a name="_Toc94288968"><span lang=EN-GB>Keynote Lecturers</span></a></h1>

<h2 align=left style='text-align:left'><span lang=EN-GB>Marco <span
class=SpellE>Vignuzzi</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Principal
Investigator, Viral Populations and Pathogenesis Unit<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin:0cm;text-align:left'><span
lang=EN-GB style='font-family:"Times New Roman",serif;color:windowtext;
mso-ansi-language:EN-GB'><span style='mso-no-proof:yes'><v:shape id="Picture_x0020_2"
 o:spid="_x0000_i1030" type="#_x0000_t75" alt="A person smiling for the camera&#10;&#10;Description automatically generated with medium confidence"
 style='width:159pt;height:158pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="AMeeGuS2022_BookOfAbstracts_20220128.fld/image004.png"
  o:href="https://ameegus-igc.github.io/images/spk1.png"/>
</v:shape></span><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Marco <span
class=SpellE>Vignuzzi</span> is a group leader at the Pasteur Institute. His
lab is investigating the rules of RNA virus evolution and population dynamics
and how these difficult to quantify aspects of infection shape viral fitness.
His insights into the complex infection cycles of many important human
pathogens come from bridging molecular virology experiments in the wet lab with
mathematical and computational approaches. Marco did his PhD at the Institute
Pasteur in the lab of Sylvie van der Werf before moving to California for his
Postdoc at the UCSF.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2 align=left style='text-align:left'><span lang=EN-GB>Rita Mateus</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Research
Group Leader at the Cluster of Excellence Physics of Life<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB;mso-no-proof:
yes'><v:shape id="Picture_x0020_10" o:spid="_x0000_i1029" type="#_x0000_t75"
 alt="A person smiling for the camera&#10;&#10;Description automatically generated with low confidence"
 style='width:228pt;height:150pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="AMeeGuS2022_BookOfAbstracts_20220128.fld/image005.jpg"
  o:title="A person smiling for the camera&#10;&#10;Description automatically generated with low confidence"
  cropleft="-441f" cropright="-516f"/>
</v:shape></span><span lang=EN-GB style='font-size:14.0pt;font-family:"Tw Cen MT",sans-serif;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Rita Mateus
a joint group leader at the Max Planck Institute of Molecular Cell Biology and
Genetics and at the Cluster of Excellence Physics of Life in Dresden, Germany.
She has always been interested in understanding how cells coordinate precise
growth and form of tissues, allowing them to become fully functional. To this
end, Rita did her PhD in Prof. Antonio Jacinto's laboratory, where she
investigated how, upon injury, the zebrafish caudal fin precisely regenerates
its shape and size, over and over, error-free. During her postdoc in the group
of Prof. Marcos Gonzalez-Gaitán, Rita turned to development to investigate how
morphogens control organ growth, using the zebrafish pectoral fin as a model.
In parallel, she became more and more interested in understanding size and
shape at the subcellular level, <span class=GramE>in particular to</span> try
to understand the physics and biology underlying structural <span class=SpellE>colors</span>,
that require the formation of specific organelles with particular morphologies.
Now, in her laboratory, Rita is pursuing these two research avenues to explore
the biophysical properties involved in controlling growth across these very
different scales<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<span lang=EN-GB style='font-size:12.0pt;font-family:"Calibri",sans-serif;
mso-ascii-theme-font:minor-latin;mso-fareast-font-family:"Times New Roman";
mso-hansi-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB;mso-fareast-language:
EN-GB;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span lang=EN-GB
style='font-size:18.0pt;font-family:"Calibri Light",sans-serif;mso-ascii-theme-font:
major-latin;mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:
major-fareast;mso-hansi-theme-font:major-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:major-bidi;color:white;mso-themecolor:background1;
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></b></p>

<h1><a name="_Toc94288969"><span lang=EN-GB>Alumni</span></a></h1>

<h2><span lang=EN-GB>Catarina Nunes</span></h2>

<p class=MsoNormal align=left style='margin:0cm;text-align:left'><span
lang=EN-GB style='font-family:"Times New Roman",serif;color:windowtext;
mso-ansi-language:EN-GB'><span style='mso-no-proof:yes'><v:shape id="Picture_x0020_4"
 o:spid="_x0000_i1028" type="#_x0000_t75" alt="A person smiling for the camera&#10;&#10;Description automatically generated with medium confidence"
 style='width:171pt;height:171pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="AMeeGuS2022_BookOfAbstracts_20220128.fld/image006.png"
  o:href="https://ameegus-igc.github.io/images/alumni1.png"/>
</v:shape></span><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Catarina
Nunes&nbsp;is an IBB alumnus from the 2016 cohort. She did her PhD work under
the supervision of <span class=SpellE>Dr.</span> <span class=SpellE>Élio</span>
<span class=SpellE>Sucena</span>, working on the regulation of immunity by
ecdysone signalling during metamorphosis, using the model organism Drosophila
melanogaster. During her work, Catarina found that the steroid signalling
responsible for regulating metamorphosis is also responsible for orchestrating
a pre-emptive immune response that might have had evolutionary implications.
Currently, she is a research associate at the Imperial College London, working
with <span class=SpellE>Dr.</span> Marc Dionne on immunometabolism and the
interplay between stress response and immunity in D. melanogaster.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Nuno Brito Santos </span></h2>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB;mso-no-proof:
yes'><v:shape id="Picture_x0020_19" o:spid="_x0000_i1027" type="#_x0000_t75"
 alt="A person sitting at a desk&#10;&#10;Description automatically generated with medium confidence"
 style='width:188pt;height:141pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="AMeeGuS2022_BookOfAbstracts_20220128.fld/image007.jpg"
  o:title="A person sitting at a desk&#10;&#10;Description automatically generated with medium confidence"/>
</v:shape></span><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Nuno Brito
Santos</span><span lang=EN-GB style='color:#239D9A;mso-ansi-language:EN-GB'> </span><span
lang=EN-GB style='mso-ansi-language:EN-GB'>is an alumnus from the 3rd edition
of the PGCD. He did his PhD with Maria João Amorim, working on the interaction
between influenza A virus and a host regulator of complement activation. His
research unveiled a novel pathway regulating lung immunopathology upon
infection. Currently, he is a postdoc at The Francis Crick Institute, working
in molecular mechanisms of SARS-CoV-2 replication in David Bauer's lab.<o:p></o:p></span></p>

<p class=Normal1><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Rafael Paiva</span></h2>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB;mso-no-proof:
yes'><v:shape id="Picture_x0020_8" o:spid="_x0000_i1026" type="#_x0000_t75"
 alt="A person smiling for the camera&#10;&#10;Description automatically generated with medium confidence"
 style='width:111pt;height:187pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="AMeeGuS2022_BookOfAbstracts_20220128.fld/image008.jpg"
  o:title="A person smiling for the camera&#10;&#10;Description automatically generated with medium confidence"/>
</v:shape></span><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Rafael
Paiva is an IBB alumnus from the class of 2016. He did his PhD in the
lymphocyte development and leukemogenesis lab working under the supervision of
Vera Martins. During this time, he studied how the thymus can maintain its
function when the supply of bone marrow progenitors is hampered and how this
promotes leukemogenesis. Specifically, he identified a population of CD4 CD8
double negative thymocytes that acquires self-renewal properties thereby
autonomously maintaining T cell development. Notwithstanding, the same
population overtime was prone to DNA damage and originated aberrant CD4 CD8
double positive thymocytes lacking productive T cell receptor </span><span
lang=EN-GB style='mso-ascii-font-family:Calibri;mso-hansi-font-family:Calibri;
mso-bidi-font-family:Calibri;mso-ansi-language:EN-GB'>β</span><span lang=EN-GB
style='mso-ansi-language:EN-GB'> rearrangements in a Notch1-dependent manner.
Rafael is currently in a transition period and will be starting in March a
postdoc in Olivier Lantz group at <span class=SpellE>Institut</span> Curie,
Paris, working on the differentiation and function of mucosal associated invariant
T cells.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<span lang=EN-GB style='font-size:12.0pt;font-family:"Calibri",sans-serif;
mso-ascii-theme-font:minor-latin;mso-fareast-font-family:"Times New Roman";
mso-hansi-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB;mso-fareast-language:
EN-GB;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span lang=EN-GB
style='font-size:18.0pt;font-family:"Calibri Light",sans-serif;mso-ascii-theme-font:
major-latin;mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:
major-fareast;mso-hansi-theme-font:major-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:major-bidi;color:white;mso-themecolor:background1;
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></b></p>

<h1><a name="_Toc94288970"><span lang=EN-GB>Long Talks</span></a></h1>

<h2><span class=SpellE><span lang=EN-GB>Abeer</span></span><span lang=EN-GB> <span
class=SpellE>Heskol</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Neurogenesis, Transcription regulation, multi-omics<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Characterization
of the Notch <span class=SpellE>cistrome</span> in vertebrate neurogenesis<o:p></o:p></span></p>

<p class=MsoNormal><span class=SpellE><span lang=EN-GB style='mso-ansi-language:
EN-GB'>Abeer</span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'> <span
class=SpellE>Heskol</span> (1, 2), <span class=SpellE>Lígia</span> Tavares (1,
2) and <span class=SpellE>Diogo</span> S. Castro (1,2)<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
i3S – Institute of Research and Innovation in Health, University of Porto,
Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>The Notch
pathway is important during the development of many tissues, where it controls
the balance between stem cell maintenance and differentiation. In the embryonic
vertebrate nervous system, the Notch pathway plays a major role regulating the
rate of neuronal differentiation of neural stem/progenitor cells during the
neurogenic phase. <span class=GramE>In spite of</span> its well characterized
function, little is still known on the regulatory rules underlying the Notch
transcriptional response. Here, I will present our ongoing work aiming at
characterizing the cis-regulation of this important pathway during
neurogenesis, by applying a genomics approach to identify Notch-responsive
regulatory regions, and associated target genes. Studies are focused on the
ventral telencephalon, a well-established model region of embryonic
neurogenesis. We are combining the genome-wide mapping of core components of
the pathway and relevant histone marks, with the interactome of active
regulatory regions and transcriptional profiling. This is applied to embryonic
brain material, or to neural stem cells cultures, where Notch signalling can be
manipulated. Integration of various data sets will address how the
cis-regulatory architecture contributes to determining the transcriptional
response to the Notch pathway. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-fareast-theme-font:minor-fareast;mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Ana <span class=SpellE>Milas</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
</span><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>Development; Cell polarity; Par proteins<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Oocyte
polarity in Drosophila is actively maintained by posterior follicle cells<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Ana <span
class=SpellE>Milas</span>, Jorge de-Carvalho and Ivo <span class=SpellE>Telley</span><o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>The
formation of the anterior–posterior body axis in Drosophila requires signalling
from posterior follicle cells (PFCs) to the stage-7 oocyte, leading to
asymmetric localization of the partitioning defective (Par) proteins. This
causes polarization of the microtubule network, which directs kinesin-dependent
transport of <span class=SpellE>oskar</span> mRNA to the posterior. Although
several studies have concluded the necessity of signalling using genetic
approaches, neither the nature of the signal nor the possible role of the PFCs
in the following stages of oogenesis are known.</span><span lang=EN-GB
style='font-size:16.0pt;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>To directly
test the role of PFCs in the maintenance of polarization, we laser ablated <span
class=SpellE>PFCs.</span> This resulted in localization of previously excluded
anterior polarity protein, Par3/Bazooka, to the posterior membrane.
Interestingly, posterior Par protein, Par1, was not excluded from this membrane
prior to the accumulation of Bazooka, suggesting that antagonism between
Bazooka and Par1 is not sufficient for their mutual exclusion. Par-1 did get
excluded from the posterior over longer timescales, which was followed by local
accumulation of microtubules.</span><span lang=EN-GB style='font-size:16.0pt;
mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-ansi-language:
EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Furthermore,
we observed that during stages of oogenesis in which Bazooka is excluded from
the posterior membrane, the oocyte is tightly connected to the PFCs, <span
class=GramE>i.e.</span> a clear boundary between the oocyte and the PFCs is not
detectable in light microscopy images. We suspect that the boundaries visibly
disappear because increased trafficking between those cells occurs, which has a
ruffling effect on the cortical membrane.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>To test
whether the contact between PFCs and the oocyte is necessary to maintain
exclusion of Bazooka, we developed an assay to mechanically detach PFCs from
the oocyte. This detachment led to the recruitment of Bazooka to the posterior
end. Incidentally, posterior localization of Bazooka coincided with the
appearance of a visible extracellular gap at the oocyte–PFC interface,
suggesting that cell-cell adhesion between germline and somatic cells is
necessary to maintain polarity.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>In
conclusion, we show for the first time that PFCs are important to maintain
polarization of Par network and the microtubule cytoskeleton in the oocyte.
This enables delivery of <span class=SpellE>oskar</span> mRNA to the posterior
of the oocyte, a process which critically depends on polarization of the
microtubule network. We propose that polarity maintenance is achieved through a
cell adhesion dependent mechanism.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Ana Teresa Mendes <span class=SpellE>Eugénio</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
transposable elements dynamics, genetic variation, evolution <o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Wolbachia
effects on Transposable Element mobilization</span><span lang=EN-GB
style='color:windowtext;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span class=SpellE><span lang=EN-GB style='mso-ansi-language:
EN-GB'>Eugénio</span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'>,
AT (1); <span class=SpellE>Beldade</span>, P (2)</span><span lang=EN-GB
style='color:windowtext;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto Gulbenkian De Ciência, Oeiras, <span class=GramE>Portugal;</span> <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
CE3C - Centre <span class=GramE>For</span> Ecology, Evolution And Environmental
Changes, FCUL University Of Lisbon, Lisbon, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Wolbachia
is a maternally transmitted bacterial endosymbiont that is widespread across
insects, <span class=GramE>arachnids</span> and nematodes. This endosymbiont
affects many aspects of host biology and can play a role in host <span
class=SpellE>defense</span>. Wolbachia confers resistance against viral
infection and affects abundance of <span class=SpellE>piRNAs</span>, which is
an important mechanism to repress transposable element (TE) activity in the
germline of eukaryotes. To test the hypothesis that Wolbachia impacts TE
mobilization in host genomes, we studied TE expression in female flies from a
panel of Drosophila melanogaster lines naturally infected with Wolbachia and
for which we have information on TE insertions. We removed Wolbachia from 25 D.
melanogaster genotypes from the Drosophila Genetic Reference Panel, to create a
panel of 25 pairs of Wolbachia-positive and Wolbachia-negative lines. Then, we
characterized Wolbachia loads in adults from the Wolbachia-positive genotypes
and measured the transcription of 14 TEs of different families in adults from
each of the Wolbachia-positive and Wolbachia negative genotypes. We found
variation in TE expression, depending on Wolbachia status, type of TE, and host
genotype. Differences in TE expression with and without Wolbachia mostly
depended on host genotype, rather than on TE identity. These data suggest that
Wolbachia may impact TE mobilization and the generation of novel genetic
variants in host genomes.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>André <span class=SpellE>Boler</span> <span class=SpellE>Cláudio</span>
da Silva Barros</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Immunology; Infection; Olfactory Receptors<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Role of
Olfactory Receptors in Infection - The case of Olfr56<o:p></o:p></span></p>

<p class=MsoNormal align=left style='margin:0cm;text-align:left;mso-line-height-alt:
11.75pt;background:white'><span lang=EN-GB style='font-family:"Arial",sans-serif;
color:#222222;mso-ansi-language:EN-GB'>André <span class=SpellE>Boler</span>
Barros<sup>1,2</sup>, <span class=SpellE>Kátia</span> Jesus<sup>1</sup>, Marc
Veldhoen<sup>3</sup>, Luís Ferreira Moita<sup>1</sup></span><span lang=EN-GB
style='font-size:11.0pt;mso-ascii-font-family:Calibri;mso-hansi-font-family:
Calibri;mso-bidi-font-family:Calibri;color:#222222;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto Gulbenkian de Ciência, 2780-156 Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
Instituto de <span class=SpellE>Tecnologia</span> <span class=SpellE>Química</span>
e <span class=SpellE>Biológica</span>, <span class=SpellE>Universidade</span>
Nova de <span class=SpellE>Lisboa</span>, 2780-157<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Oeiras,
Portugal<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(3)
Instituto de <span class=SpellE>Medicina</span> Molecular João Lobo Antunes,
1649-028 <span class=SpellE>Lisboa</span>, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Olfactory
receptors are a key component on detecting and reacting to the environment.
Apart from this role, these G-Protein Coupled Receptors are present in
non-sensory organism and cells, as well as involved in non-sensory pathways.
Specifically in the context of infection, these receptors have been showed to
be expressed in immune cells. They are also involved in sensing
pathogen-metabolites, such as acetate.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Transcriptomic
data has shown some Olfactory Receptors are highly regulated upon infection.
The one with the most substantial change is Olfr56. This receptor has a strong
upregulation, both on mRNA and on protein levels, during the 24h period post-
infection. In addition, this upregulation seems to be characteristic to several
immune cell types. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Using a
transgenic mouse with a non-functional version of the protein, we have tested
several in vivo infection models. On this context, only the Cecum Ligation and
Puncture (CLP) model has shown interesting results, with increased survival for
the Olfr56-/-. Within vitro stimulation with (LPS), we have observed a
gender-variable regulation of proinflammatory cytokines, such as IL1b. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>The results
of this project can further improve our understanding of the role of olfactory
receptors in immune response. Plus, considering the pharmacological importance
of GPCRs, this project may open new avenues on infection treatment.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Carla Filipa <span class=SpellE>Simões</span> Henriques</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Neuroscience; Behaviour; Genetics<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Functional
specialization of social and asocial learnings in Drosophila melanogaster<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Carla
Simões-Henriques1; Joana Marcos1; <span class=SpellE>Élio</span> Sucena1; Maria
L. Vasconcelos2; Susana A. M. Varela1,3*; Rui F. Oliveira1,2,3*</span><span
lang=EN-GB style='font-family:"Times New Roman",serif;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto Gulbenkian de Ciência, Oeiras, <span class=GramE>Portugal;</span> <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
<span class=SpellE>Champalimaud</span> Centre for the Unknown, <span
class=SpellE>Lisboa</span>, <span class=GramE>Portugal;</span> <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(3)
ISPA-<span class=SpellE>Istituto</span> <span class=SpellE>Universitário</span>,
<span class=SpellE>Lisboa</span>, <span class=GramE>Portugal;</span> <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>*PI
and co-PI of the <span class=SpellE>SocialFly</span> Project, respectively<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Social and asocial
learnings are required to cope with the complexity of the environment. They are
behaviourally distinct. However, whether they use a single (general-purpose) or
distinct (<span class=GramE>special-purpose</span>) cognitive mechanisms
remains elusive. A general-purpose mechanism comprises a single set of genes
regulating the same biochemical pathways and neural circuits, while a
special-purpose mechanism comprises two sets of genes, possibly affecting
different biochemical pathways and/or neural circuits.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>To address
this question, we firstly tested social and asocial learning abilities in 40
lines of the DGRP, a panel of several isogenic sequenced lines that together
represent the genetic variation of a natural Drosophila population. For asocial
learning, individual flies had to learn to avoid an oviposition-site where they
associated an odour (conditioned stimulus, CS) with a bitter taste
(unconditioned stimulus, US). For social learning, flies had to learn the
oviposition-site choice of a group of conspecifics previously exposed to the CS
and the US. Secondly, we performed a Genome-Wide Association Study – a
statistical correlation between the genetic variants in the DGRP and each
learning phenotype. We obtained two completely different sets of candidate
genes for social and asocial learnings. Thirdly, to functionally validate the
role of each gene on each learning phenotype, we used GAL4/RNAi-UAS lines to
knockdown the genes. We found genes that affect both learnings <span
class=GramE>and also</span> cases of specialization.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>This is the
first experimental evidence supporting the special-purpose cognitive mechanism
hypothesis. Future work should focus on the extent of this specialization,
concerning the biochemical pathways and the neural circuitry.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Financial
support was granted by the Portuguese Foundation for Science and Technology
through the Project “<span class=SpellE>TheSocialFly</span>”
(PTDC/BIA-COM/31887/2017), Carla Henriques PhD scholarship (PD/BD/138178/2018)
and Joana Marcos fellowship (203/BI/19). Financial support was also granted by
ASAB, the Association for the Study of Animal Behaviour.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Christian <span class=SpellE>Diwo</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Membrane protein biogenesis, viral infection<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>How
membrane proteins encode their fate to reach the membrane - the determinants of
membrane protein translocation route selection at the ER<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Christian <span
class=SpellE>Diwo</span>, Colin <span class=SpellE>Adrain</span></span><span
lang=EN-GB style='font-family:"Times New Roman",serif;mso-ansi-language:EN-GB'>
and </span><span lang=EN-GB style='mso-ansi-language:EN-GB'>Maria-João Amorim</span><span
lang=EN-GB style='font-family:"Times New Roman",serif;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian De Ciência, Oeiras, <span class=GramE>Portugal;</span> <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Biological
membranes are functionalized through the dense population with membrane
spanning proteins which is a fundamental requisite for life. In modern
eukaryotic cells around 1/4 of all proteins synthesized in the cytosol are
inserted into the endoplasmic reticulum (ER) membrane by broadly conserved
protein complexes. The Sec61 protein conducting channel is often the <span
class=SpellE>center</span> piece in various assemblages that facilitate the
membrane insertion or translocation of <span class=SpellE>physico</span>-chemical
diverse membrane proteins but recently more ER membrane components have been
found to act as alternative translocation routes. Many molecular observations
in the chain of events <span class=GramE>from protein translation,</span> to ER
membrane targeting of translating ribosomes, and finally protein translocation
kinetics contribute to our current understanding of membrane protein
biogenesis, however it remains unclear how a specific protein influences its
own translocation route. This is important as membrane protein biogenesis is
inherently error prone and may lead to disease when quality control mechanisms
are failing. I am developing biochemical and proteomic tools with which I aim
to answer how protein import at the ER is concerted by its various
translocation routes and impacted by conditions like viral insults. We observe
that a small viral membrane protein M2 of the influenza A virus (IAV) is
dependent on an alternative membrane insertion pathway, the endoplasmic
reticulum membrane protein complex (EMC). Previous reports show that the
M2-like membrane protein architecture can be dependent on either Sec61 or EMC
processing, which allows us to rationally design mutants that switch their
dependencies. <span class=GramE>In order to</span> gain statistical power in
discriminating protein features that determine translocation route I am
developing a labelling approach based on the ER luminal expression of a
promiscuous biotin ligase. By labelling imported proteins with <span
class=GramE>biotin</span> we will be able to interrogate protein import at the
ER on a proteome scale under conditions where a certain translocation route is
ablated or monitor protein import changes during IAV or SARS-<span
class=SpellE>CoV</span> 2 infection.</span><span lang=EN-GB style='font-family:
"Times New Roman",serif;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=Normal1><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Massimo <span class=SpellE>Amicone</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
</span><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-ansi-language:EN-GB'>SARS-CoV-2; Evolution; Mutation rates<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Mutation
rate of SARS-CoV-2 and emergence of mutators during experimental evolution.<o:p></o:p></span></p>

<p class=MsoNormal><span class=SpellE><span lang=EN-GB style='mso-ansi-language:
EN-GB'>Amicone</span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'>,
M (1); Borges, V (2); Alves, MJ (3); Isidro, J (2); <span class=SpellE>Zé-Zé</span>,
L (3,4); Duarte, S (5); Vieira, L (5,6); <span class=SpellE>Guiomar</span>, R
(7); Gomes, JP (2); <span class=SpellE>Gordo</span>, I (1).</span><span
lang=EN-GB style='font-family:"Times New Roman",serif;color:windowtext;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
IGC – Instituto Gulbenkian de Ciência, Oeiras, <span class=GramE>Portugal;</span>
<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
INSA – Bioinformatics Unit, Department of Infectious Diseases, National
Institute of Health <span class=SpellE>Doutor</span> Ricardo Jorge, Lisbon, <span
class=GramE>Portugal;</span> <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(3)
INSA – Centre for Vectors and Infectious Diseases Research, Department of
Infectious Diseases, National Institute of Health <span class=SpellE>Doutor</span>
Ricardo Jorge, <span class=SpellE>Águas</span> de Moura, <span class=GramE>Portugal;</span>
<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(4)
<span class=SpellE>BioISI</span> – Biosystems &amp; Integrative Sciences
Institute, Faculty of Sciences, University of Lisbon, <span class=GramE>Portugal;</span>
<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(5)
INSA – Innovation and Technology Unit, Department of Human Genetics, National
Institute of Health <span class=SpellE>Doutor</span> Ricardo Jorge, Lisbon, <span
class=GramE>Portugal;</span> <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(6)
<span class=SpellE>ToxOmics</span> – Centre for <span class=SpellE>Toxicogenomics</span>
and Human Health, Genetics, Oncology and Human Toxicology, Nova Medical <span
class=SpellE>School|Faculdade</span> de <span class=SpellE>Ciências</span> <span
class=SpellE>Médicas</span>, <span class=SpellE>Universidade</span> Nova de <span
class=SpellE>Lisboa</span>, Lisbon, <span class=GramE>Portugal;</span> <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(7)
INSA – National Reference Laboratory for Influenza and other Respiratory
Viruses, Department of Infectious Diseases, National Institute of Health <span
class=SpellE>Doutor</span> Ricardo Jorge, Lisbon, Portugal.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>To
understand how organisms evolve, it is fundamental to study how mutations
emerge and establish. Here, we estimated the rate of mutation accumulation of
SARS-CoV-2 in vitro and investigated the repeatability of its evolution when
facing a new cell type but no immune or drug pressures. We performed experimental
evolution with two strains of SARS-CoV-2, one carrying the originally described
spike protein (CoV-2-D) and another carrying the D614G mutation that has spread
worldwide (CoV-2-G). After 15 passages in Vero cells and whole genome
sequencing, we characterized the spectrum and rate of the emerging mutations
and looked for evidence of selection across the genomes of both strains. From
the frequencies of the mutations accumulated, and excluding the genes with
signals of selection, we estimate a spontaneous mutation rate of 1.3x10-6 ±
0.2x10-6 per-base per-infection cycle (mean across both lineages of SARS-CoV-2
± 2SEM). We further show that mutation accumulation is larger in the CoV-2-D
lineage and heterogeneous along the genome, consistent with the action of
positive selection on the spike protein. We also observe the emergence of
mutators in the CoV-2-G background, likely linked to mutations in the
RNA-dependent RNA polymerase and/or in the error-correcting exonuclease
protein. These results provide valuable information on how spontaneous
mutations emerge in SARS-CoV-2 and on how selection can shape its genome
towards adaptation to new environments.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Patricia Mónica Silva Costa </span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Plant biology, root development <o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>SnRK1 dual
function in lateral root development<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Monica
Costa, Tom <span class=SpellE>Beeckman</span>, Elena <span class=SpellE>Baena</span>-Gonzalez<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Plants
for Life PhD program ITQB Nova / Root Development lab - VIB / Plant Stress <span
class=SpellE>Signaling</span> group - IGC <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>One major
strategy of plants to adapt to highly heterogeneous and often challenging
environments is to modify root architecture. To this end, dedicated <span
class=SpellE>signaling</span> pathways integrate environmental cues with
hard-wired developmental programs into a unified response that tailors growth
in accordance with the environment. How this integration occurs is poorly
understood but has been proposed to involve carbon <span class=SpellE>signaling</span>
through the Sucrose non-fermenting Related Kinase 1 (SnRK1). In agreement,
SnRK1 knockdown mutants show defective repression of lateral root (LR)
formation in response to ABA (a major stress hormone), suggesting that SnRK1 is
important for readjusting root architecture in situations of stress. To
elucidate the mechanisms underlying SnRK1 effect, we are currently undertaking
several complementary strategies. Firstly, we aim to identify the stage of LR
development targeted by SnRK1. LRs form by proliferation of specific pericycle
cells that differentiate into a new organ. This process can be divided into 4
steps: founder cell specification, initiation, primordium formation, emergence,
and meristem activation. ABA has been implicated in several of these
developmental steps. Phenotyping of constitutive SnRK1 mutants and an inducible
loss-of-function line suggest that, under normal conditions, SnRK1 stimulates
early development of LR, prior to initiation, whilst in the presence of ABA,
SnRK1 inhibits post emergence LR elongation. We have also uncovered an impact
of SnRK1 in primary root growth, in what seems to be a different role to the
one observed in LRs. Altogether, our results highlight a major role for SnRK1
in shaping root architecture.<o:p></o:p></span></p>

<p class=Normal1><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span class=SpellE><span lang=EN-GB>Patrícia</span></span><span lang=EN-GB>
Rodrigues Duarte</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Embryonic development<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Evaluating
the switch in cellular competence during head to trunk transition<o:p></o:p></span></p>

<p class=MsoNormal><span class=SpellE><span lang=EN-GB style='mso-ansi-language:
EN-GB'>Patrícia</span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'>
Duarte, André Dias, Ana <span class=SpellE>Nóvoa</span>, <span class=SpellE>Moisés</span>
<span class=SpellE>Mallo</span><o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Development
of vertebrate embryos is characterized by early formation of the anterior
tissues and sequential axial extension at their posterior end, first by the
activity of the primitive streak and then of the tail bud. Head and trunk
development are significantly different, and this transition involves a change
in the capacity of cells to respond to <span class=GramE>particular gene</span>
networks when entering the trunk formation stage. Genetic experiments indicate
that retinoic acid is crucial during this transition phase, becoming
dispensable once trunk formation has started. Herein, we aim to understand the
mechanisms behind the switch in cell competence during head to trunk transition
and the role that retinoic acid plays in this process. For this, we performed a
genome wide chromatin accessibility assay to identify regulatory regions
involved in this transition, followed by their functional assessment using
transgenic and genome editing strategies in mouse embryos. This approach
allowed us to identify a novel enhancer regulating the expression of Hoxa5.
Moreover, transcription factor footprinting analysis revealed the presence of a
VDR (Vitamin D Receptor) binding site involved in the regulation of this
enhancer.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Renato José Melo <span class=SpellE>Sosua</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
</span><span lang=EN-GB style='mso-ansi-language:EN-GB;mso-fareast-language:
EN-US'>Neurobiology, Dopamine, Learning<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Towards an
understanding of parallel learning systems during decisions in a dynamic value
landscape<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Renato
Sousa (1,2), Rui F. Oliveira (1, 2, 3), Joseph J. Paton (2)<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto Gulbenkian de Ciência<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
<span class=SpellE>Champalimaud</span> Neuroscience Programme<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(3)
ISPA - <span class=SpellE>Instituo</span> <span class=SpellE>Universitário</span>,
<span class=SpellE>Lisboa</span>, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Animals
strive to maximize returns by either acquiring rewards or avoiding punishments.
Accurately predicting what actions best lead to desired outcomes is thus a
fundamental skill, well-grounded in the function of midbrain dopamine (DA)
neurons. However, DA neurons are embedded in a circuit architecture that
constrains the type of information they have access to and, consequently, their
function. Along the striatum, projections from DA neurons converge with
cortico-striatal synapses to form a tripartite unit where values of actions are
materialized. Though, it remains poorly understood whether the role of this
tripartite unit for learning depends on striatal anatomy or not.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>In this project,
we address these parallel learning systems by recording and perturbing
individual components of this unit in functionally well-defined striatal
domains. We show that mice performing a probabilistic task use their recent
trial history to determine what actions are most valuable and adapt their
choices. We observe that DA terminals in the dorsolateral striatum (DLS) scale
their responses with a relative value gradient for choices that result in a
reward. Lastly, we also demonstrate that optogenetic activation of indirect
pathway neurons (<span class=SpellE>iMSNs</span>) in the tail of the striatum
(TS) increases the probability of choosing the ipsilateral option.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>In natural
settings, mice face complex decisions that result from the multidimensionality
of the environment they inhabit. To better approximate our task to a more
ethological circumstance, we will complement it with an <span class=SpellE>odor</span>
compound derived from the urine of a naturally occurring predator. We expect
this additional stimulus of opposing value will enhance the contrast between
potential action outcomes and, therefore, invoke activity in DA systems that
best resembles the patterns underlying learning in naturally occurring
scenarios.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span class=SpellE><span lang=EN-GB>Sónia</span></span><span lang=EN-GB>
Maria <span class=SpellE>Leite</span> Pereira <o:p></o:p></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-fareast-font-family:Calibri;
mso-ansi-language:EN-GB'>Keywords: Malaria, Immunity, Infection, Disease<span
style='mso-spacerun:yes'>  </span><o:p></o:p></span></p>

<p class=MsoNormal><span class=SpellE><span lang=EN-GB style='mso-fareast-font-family:
Calibri;mso-ansi-language:EN-GB'>Sónia</span></span><span lang=EN-GB
style='mso-fareast-font-family:Calibri;mso-ansi-language:EN-GB'> Pereira <i
style='mso-bidi-font-style:normal'>et al., <o:p></o:p></i></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-fareast-font-family:Calibri;
mso-ansi-language:EN-GB'>Instituto de <span class=SpellE>Tecnologia</span> <span
class=SpellE>Química</span> e <span class=SpellE>Biológica</span> António
Xavier, <span class=SpellE>Universidade</span> Nova de <span class=SpellE>Lisboa</span>
(ITQB - NOVA), Oeiras, Portugal<span style='mso-spacerun:yes'>  </span><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:Calibri;
mso-ansi-language:EN-GB'>Malaria is still a major health concern in the XXI
century. Being responsible for about half <span class=GramE>a<span
style='mso-spacerun:yes'>  </span>million</span> deaths every year, malaria´s incidence,
stretching across vast geographic areas, puts<span style='mso-spacerun:yes'> 
</span>nearly half of the world’s population at risk. In the mammalian host,
the life cycle <span class=GramE>of<span style='mso-spacerun:yes'> 
</span>Plasmodium</span> encompasses two distinct replicative stages. The
first, considered clinically <span class=GramE>silent<span
style='mso-spacerun:yes'>  </span>despite</span> obligatory, occurs in the
liver, where the parasite invades and replicates within the<span
style='mso-spacerun:yes'>  </span>hepatocyte, i.e., the liver-stage (LS); and
the second, initiated once the liver-born parasitic<span
style='mso-spacerun:yes'>  </span>progeny reaches the blood and infects the red
blood cell (RBC), i.e., the blood-stage (BS),<span style='mso-spacerun:yes'> 
</span>which is associated with the emergence of clinical manifestations and
disease. During <span class=GramE>malarial<span style='mso-spacerun:yes'> 
</span>infections</span>, parasite density results from the interplay between
immunity against Plasmodium<span style='mso-spacerun:yes'>  </span>and
virulence mechanisms of the parasite. Overall, the balance between parasite <span
class=GramE>elimination<span style='mso-spacerun:yes'>  </span>and</span>
persistence has a major impact on the clinical outcome of infection, with
parasite burden<span style='mso-spacerun:yes'>  </span>being currently the
best-known correlate of disease severity following Plasmodium infections<span
style='mso-spacerun:yes'>  </span>in humans. We took the view that the LS of
infection, albeit silent, may influence <span class=GramE>parasite<span
style='mso-spacerun:yes'>  </span>virulence</span> and/or host-driven immune
mechanisms aiming at parasite elimination, ultimately<span
style='mso-spacerun:yes'>  </span>precluding effective control of parasite
density. By using a rodent model of <span class=GramE>interrupted<span
style='mso-spacerun:yes'>  </span>hepatic</span> infection followed by an
infectious BS, we have now established that this is the case.<span
style='mso-spacerun:yes'>  </span>The establishment of Plasmodium parasites in
the liver prior to a BS infection is <span class=GramE>correlated<span
style='mso-spacerun:yes'>  </span>with</span> decreased removal of parasitized
RBCs by the spleen, and effect that is, at least partially,<span
style='mso-spacerun:yes'>  </span>dependent on T cells and/or B cells. In the
future, we intend to unveil the mechanistic <span class=GramE>basis<span
style='mso-spacerun:yes'>  </span>of</span> this critical aspect of
host-Plasmodium interactions where the non-symptomatically LS of<span
style='mso-spacerun:yes'>  </span>infection impacts on parasite virulence and,
ultimately, on the pathological outcome of<span style='mso-spacerun:yes'> 
</span>infection. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Temitope Wilson <span class=SpellE>Ademolue</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Infectious Diseases, Metabolism<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>A role of <span
class=SpellE>Heme</span> in Immunity against Malaria<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Temitope
Wilson <span class=SpellE>Ademolue</span> (1), Susana Ramos (1), Elisa <span
class=SpellE>Jentho</span> (1,2), Qian Wu (1), Joel Guerra (2), Rui Martins (1),
Gil Pires (1), Sebastian Weis (2,3,4), Ana Rita Carlos (1), <span class=SpellE>Inês</span>
<span class=SpellE>Mahú</span> (1), Elsa <span class=SpellE>Seixas</span> (1),
Denise Duarte (5), Fabienne Rajas (6), <span class=SpellE>Sílvia</span> Cardoso
(1), António G. G. Sousa (1), <span class=SpellE>Jingtao</span> <span
class=SpellE>Lilue</span> (1), Gilles <span class=SpellE>Mithieux</span> (6),
Fátima Nogueira (5) and Miguel P. Soares (1)<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto Gulbenkian de Ciência, Oeiras, Portugal.<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
Department of <span class=SpellE>Anesthesiology</span> and Intensive Care
Medicine, Jena University Hospital, Friedrich-Schiller-University, Jena,
Germany.<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(3)
Institute for Infectious Disease and Infection Control, University Hospital
Jena, Germany. <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(4)
<span class=SpellE>Center</span> for Sepsis Control and Care, Jena University,
Germany.<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(5)
Global Health and Tropical Medicine, Instituto de <span class=SpellE>Higiene</span>
e <span class=SpellE>Medicina</span> Tropical, <span class=SpellE>Universidade</span>
NOVA de <span class=SpellE>Lisboa</span>, <span class=SpellE>Lisboa</span>,
Portugal.<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(6)
<span class=SpellE>Université</span> Claude Bernard Lyon, INSERM, France.<o:p></o:p></span></p>

<p class=MsoNormal><span class=SpellE><span lang=EN-GB style='mso-ansi-language:
EN-GB'>Hypoglycemia</span></span><span lang=EN-GB style='mso-ansi-language:
EN-GB'> is a clinical hallmark of severe malaria, the often-lethal presentation
of Plasmodium falciparum infection of humans. Here we report that
malaria-associated <span class=SpellE>hypoglycemia</span> emerges from a
hypometabolic-based <span class=SpellE>defense</span> strategy whereby the
infected host reduces blood glucose levels. This occurs via a coordinated response
elicited by labile <span class=SpellE>heme</span>, a <span class=SpellE>byproduct</span>
of <span class=SpellE>hemolysis</span>, which induces anorexia and represses
hepatic glucose production (HGP). While protective against unfettered
immune-mediated inflammation, organ damage and <span class=SpellE>anemia</span>,
repression of HGP can progress towards <span class=SpellE>hypoglycemia</span>
when sustained over time, compromising host energy expenditure and
thermoregulation. This hypometabolic state arrests the development of asexual
stages of Plasmodium spp., which undergo pyknosis, associated with
mitochondrial dysfunction. In response, Plasmodium activates a transcriptional
program reducing virulence and inducing sexual differentiation towards
transmissible gametocytes. In conclusion, malaria-associated <span
class=SpellE>hypoglycemia</span> represents a trade-off of an evolutionarily
conserved <span class=SpellE>defense</span> strategy restricting Plasmodium
spp. from accessing host-derived glucose and balancing parasite virulence and
transmission.<br clear=all style='mso-special-character:line-break;page-break-before:
always'>
<o:p></o:p></span></p>

<h1><a name="_Toc94288971"><span lang=EN-GB>Short Talks</span></a><span
lang=EN-GB> </span></h1>

<h2><span lang=EN-GB>Camila Veludo Ramos</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Immunology, Cell biology<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>IL-7r is
not required at the DN3e in steady state <span class=SpellE>thymopoiesis</span><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Camila
Ramos and Vera Martins<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de <span class=SpellE>Ciênciia</span>, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>T
lymphocyte differentiation requires interleukin 7 (IL-7)/IL-7 receptor (IL-7r) <span
class=SpellE>signaling</span> to ensure the expansion of T cell precursors
prior to key selection events. IL-7r deficient mice are <span class=SpellE>lymphopenic</span>,
and immature thymocytes lacking IL-7r have severe proliferation and survival
defects. This compromises the generation of double-negative 3 early (DN3e)
cells, but whether DN3e require IL-7r <span class=SpellE>signaling</span> has
yet to be addressed. Here, by conditionally deleting the IL-7r locus at the
DN3e, we showed that DN3e can respond to increase in IL-7 availability by
proliferating. However, in a competitive setting, IL-7r deficient DN3e achieved
a cellularity <span class=GramE>similar to</span> that of wild-type DN3e, with
comparable levels of apoptosis and proliferation. Taken together, our results
suggest that DN3e do not respond to IL-7 unless its uptake by other T cell
precursors is reduced.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Catarina Sofia Rodrigues do Carmo</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Cell Biology<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Lodestar,
a protein with a dual role in mitotic transcription inactivation and sister
chromatid resolution<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Catarina
Carmo, João Coelho, <span class=SpellE>Abha</span> <span class=GramE>Bose</span>
and Raquel Oliveira<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Rua da Quinta Grande, 2780-156 Oeiras, Portugal</span><span
lang=EN-GB style='font-family:"MS Gothic";mso-bidi-font-family:"MS Gothic";
mso-ansi-language:EN-GB'> </span><span lang=EN-GB style='mso-ansi-language:
EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Mitosis is
essential for life. This process requires drastic changes in chromosome
structure that parallels with a major switch off in transcriptional activity.
While it has long been assumed that increased compaction would render mitotic
chromatin inaccessible for transcription, the interplay between mitotic
chromosome organization and transcription termination remains elusive.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>While
looking for novel genetic interactions with chromosome assembly factors, we
uncovered a poorly characterized helicase-like protein – TTF2 (transcription
termination factor 2) / <span class=SpellE>Lds</span> (lodestar in Drosophila
melanogaster).<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>We found
that <span class=SpellE>Lds</span> is predominantly cytoplasmic during
interphase and localizes to mitotic chromosomes a few seconds prior to Nuclear
Envelope Breakdown, suggesting it may be critical for mitosis-specific
chromatin events. Accordingly, absence of <span class=SpellE>Lds</span> results
in severe segregation errors (1), mostly characterized by chromatin bridges
during anaphase. However, how <span class=SpellE>Lds</span> ensures faithful
chromosome segregation remains unknown.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Given that
absence of <span class=SpellE>Lds</span> results in severe chromatin bridges,
we hypothesize that it aids in the process of sister chromatid resolution.
Accordingly, we found that <span class=SpellE>Lds</span> interacts with the
major enzyme that resolves sister chromatid intertwines - Topoisomerase II –
and localizes to chromatin bridges at mitotic exit. These findings suggest <span
class=SpellE>Lds</span> cooperates with Topoisomerase II to facilitate sister
chromatid resolution.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Strikingly,
previous studies support that <span class=SpellE>Lds</span> holds transcription
termination activity in vitro (2), which would indicate that the observed
segregation defects could be (at least partially) attributed to impairment in
transcription termination. Indeed, we show that <span class=SpellE>Lds</span>
is required for timely release of transcripts from chromatin at mitotic entry.
Using a system to monitor ongoing transcription in real time (3) we see that in
embryos with reduced levels of <span class=SpellE>Lds</span> retain transcripts
at mitotic chromatin, despite normal mitotic progression and chromosome compaction.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>We propose
that <span class=SpellE>Lds</span> has a dual role during mitosis, by ensuring
prompt inactivation of transcription and cooperating in sister chromatid
resolution. Ongoing work is aiming to dissect how these two functions are
mechanistically linked and the contribution of these two processes for mitotic
fidelity.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Hugo Manuel <span class=SpellE>Condessa</span> Barreto</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Evolution, Microbiology, Antibiotics<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Gut
mélange à trois: fluctuating selection modulated by microbiota, host immune
system, and antibiotics<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Hugo C.
Barreto, Beatriz Abreu and Isabel <span class=SpellE>Gordo</span><o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Rua da Quinta Grande 6, 2780-156 Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Iron is
critical in host-microbe interactions, and its availability is tightly
regulated in the mammalian gut. Antibiotics and inflammation can perturb iron
availability in the gut, which could alter host-microbe interactions. Here, we
show that an adaptive allele of <span class=SpellE>iscR</span>, a major
regulator of iron homeostasis of Escherichia coli, is under fluctuating
selection in the mouse gut. In vivo competitions in immune-competent,
immune-compromised, and germ-free mice reveal that the selective pressure on an
<span class=SpellE>iscR</span> mutant E. coli is modulated by the presence of
antibiotics, the microbiota, and the immune system. In vitro assays show that
iron availability is an important mediator of the <span class=SpellE>iscR</span>
allele fitness benefits or costs. We identify Lipocalin-2, a host’s immune
protein that prevents bacterial iron acquisition, as a major host mechanism
underlying fluctuating selection of <span class=SpellE>iscR</span>. Our results
provide a remarkable example of strong fluctuating selection acting on
bacterial iron regulation in the mammalian gut. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Marco António Dias <span class=SpellE>Louro</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Cell Biology, Evolution<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Unravelling
the population dynamics of centrosome amplification<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Marco
António Dias <span class=SpellE>Louro</span>, Mónica Bettencourt-Dias, Claudia
Bank<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Understanding
the causes and consequences of organelle copy number variation is a fundamental
problem in biology. For certain subcellular structures, such as centrioles,
copy number is near invariant in healthy proliferating cells, ranging from two
to four depending on the cell cycle stage. In cancer cells, centriole number
tends to be higher, a phenomenon termed centrosome amplification, but widely
heterogeneous. How this intra- and intercellular variation is determined, and
how it affects cellular evolution, remain poorly understood.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>First, I
will present how centriole number variation can be generated in equilibrium
conditions. Using mathematical models inspired in population genetics, we
showed that a balance between centriole (over)production and negative selection
can explain a variety of empirical centriole number distributions.
Surprisingly, most of the best-fitting models assume the existence of
overduplication events and negative selection against cells with any number of
extra centrioles. This suggests that selection can efficiently eliminate cells
with centrosome amplification.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Secondly, I
will discuss how cells evolve when centriole number is driven out of
equilibrium. Using cell lines carrying a doxycycline-inducible overexpression
system for Plk4, a key regulator of centriole biogenesis, we generated
populations of high and low initial frequencies of cells with extra centrioles.
These populations suffered a proliferative disadvantage compared to their
non-induced counterparts, which increased with the initial frequency of cells
with extra centrioles. We grew them for two months under continuous induction
and observed that this proliferative disadvantage was seemingly overcome after
two to three weeks. Interestingly, cells with extra centrioles were
progressively lost in the population in which they occurred at high initial
frequencies but not in the one with low initial frequencies.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Thus,
despite cells with centrosome amplification being counter-selected, they can be
maintained in a proliferating population if they are relatively rare, which
could explain how they arise in tumours and potentially drive cancer evolution.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Tânia Filipa Teixeira Paulo</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Evolution, Innate immunity<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Identifying
mechanisms underlying adaptation against oral bacterial infection in D.
melanogaster.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Tânia F.
Paulo and <span class=SpellE>Élio</span> <span class=SpellE>Sucena</span><o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>In natural
environments, pathogens exert strong selective pressures on hosts, which in
turn counteract by deploying a vast array of defence strategies. For Drosophila
melanogaster these defences encompass behavioural, <span class=GramE>developmental</span>
and physiological dimensions and although much is known about immunity in this
model, gaps remain on how evolution can act on its response mechanisms.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>In previous
work, we selected an outbred population of D. melanogaster against oral
infection with its natural pathogen Pseudomonas <span class=SpellE>entomophila</span>.
After just 10 generations of selection, survival upon infection had reached
approximately 100% of the population and this increased survival was
maintained, even under relaxed selection, for over 80 generations. <br>
We initially characterized bacterial loads dynamics upon infection at different
timepoints and found that the Evolved population was clearing the infection
faster and more efficiently than its Control, which points to an acquired
higher resistance capability. <o:p></o:p></span></p>

<p class=MsoNormal><span class=GramE><span lang=EN-GB style='mso-ansi-language:
EN-GB'>In order to</span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'>
understand which mechanisms were responsible and whether we were dealing with a
local or general response, we performed RNA-<span class=SpellE>seq</span>
analysis on samples of both populations. Analysis of these data identified gut
traffic and peritrophic matrix action, as well as earlier control of immune
response as the main players in the adapted population’s response. <span
class=GramE>Consequently</span> we are currently validating our main gene
candidates using RNAi and hope to shed light on the relative importance of each
mechanism to the overall increased survival response. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Temitope <span class=SpellE>Akhigbe</span> <span
class=SpellE>Etibor</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Virology, Cell biology, Biophysics<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Biophysics
and composition of Influenza A viral inclusion<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Temitope <span
class=SpellE>Akhigbe</span> <span class=SpellE>Etibor</span> (1), <span
class=SpellE>Sindhuja</span> Sridharan (2), Daniela <span class=SpellE>Brás</span>
(1), Isabelle Berger (2), Victor Mello (1), Filipe Ferreira (1), <span
class=SpellE>Sílvia</span> Vale-Costa (1), Mikhail <span class=SpellE>Savitski</span>
(2) and Maria João Amorim (1)<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Cell Biology of Viral Infection Lab, Instituto Gulbenkian de Ciência, Oeiras,
Portugal<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
EMBL, Heidelberg, Germany<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Influenza A
virus is a negative sense RNA virus with segmented genome packed in the form of
viral ribonucleoproteins. Upon infection, these segmented <span class=SpellE>vRNPs</span>
are released into the host cell, and at late infection stage are found in
cytoplasmic <span class=SpellE>punctae</span> called viral inclusions. These
inclusions were hypothesized as the site of genome assembly since all eight 8 <span
class=SpellE>vRNP</span> segments colocalize within it. We for the first time
showed that viral inclusion formation precedes intersegment <span class=SpellE>vRNP-vRNP</span>
interactions. Furthermore, using correlative light and electron microscopy
(CLEM), we demonstrated that viral inclusions are not delimited by membrane but
show hallmarks of liquid-like biomolecular condensates. Our solubility and
biophysical analysis indicated that liquid inclusions which are
component-dependent, mature with increasing concentration of cytoplasmic <span
class=SpellE>vRNPs</span> and governed by temperature-dependent thermodynamics.
Importantly, we demonstrated that viral inclusions are innately liquid and can
be modified to a solid like inert state by subjecting them to a drug-induced
phase transition. This resulted in a stabilized free energy through steric <span
class=SpellE>vRNP-vRNP</span> interlocking thereby specifically altering
inclusion material properties of in vivo and ex vivo models of infection,
without affecting other viral induced or cellular condensates.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<span lang=EN-GB style='font-size:12.0pt;font-family:"Calibri",sans-serif;
mso-ascii-theme-font:minor-latin;mso-fareast-font-family:"Times New Roman";
mso-hansi-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
color:black;mso-themecolor:text1;mso-ansi-language:EN-GB;mso-fareast-language:
EN-GB;mso-bidi-language:AR-SA'><br clear=all style='mso-special-character:line-break;
page-break-before:always'>
</span>

<p class=MsoNormal align=left style='text-align:left'><b><span lang=EN-GB
style='font-size:18.0pt;font-family:"Calibri Light",sans-serif;mso-ascii-theme-font:
major-latin;mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:
major-fareast;mso-hansi-theme-font:major-latin;mso-bidi-font-family:"Times New Roman";
mso-bidi-theme-font:major-bidi;color:white;mso-themecolor:background1;
mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></b></p>

<h1><a name="_Toc94288972"><span lang=EN-GB>Posters</span></a></h1>

<h2><span lang=EN-GB>Afonso Machado Mendes</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
HIV-1 Assembly, Super-Resolution Microscopy, Single Particle Analysis<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Unveiling
the temporal and spatial hierarchy of HIV-1 viral assembly and its restriction
by host innate immunity<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Afonso
Mendes (1) and Ricardo Henriques (1) (2)<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
MRC-Laboratory for Molecular Cell Biology. University College London, London,
UK<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Human Immunodeficiency
Virus (HIV) infection affects approximately 38 million people worldwide and
caused 690 000 deaths in 2019. The structure and composition of HIV have been
well-described using electron microscopy and crystallography. However, these
approaches mainly focused on providing a collective average model of viral
strains. An enormous knowledge gap remains in understanding how viral
replication varies within a cellular population and how an individual cell
yields locally and temporally distinct viruses in their morphology and
composition. Understanding this variability is critical to elucidate why a
small percentage of produced viruses are replication-competent and to further
our knowledge on the mechanistic basis of infection. Recent advancements in imaging
technology provide the capacity to follow viral morphogenesis at the
near-molecular scale within living cells. This project aims to harness the
power of super-resolution microscopy and machine-learning to describe these
processes quantitatively.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Ana Sofia Soares Mendes</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Antimicrobial; Carbon Monoxide-Releasing Molecules<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Carbon
Monoxide Releasing Molecules against pathogens<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Sofia S.
Mendes (1), Joana T. Marques (1), Diana <span class=SpellE>Fontinha</span> (2),
Diana Moita2, Carlos C. <span class=SpellE>Romão</span> (1), Miguel <span
class=SpellE>Prudêncio</span> (2) and <span class=SpellE>Lígia</span> M.
Saraiva (1)<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto de <span class=SpellE>Tecnologia</span> <span class=SpellE>Química</span>
e <span class=SpellE>Biológica</span> António Xavier, Av. da <span
class=SpellE>República</span>, 2780-157 Oeiras <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
Instituto de <span class=SpellE>Medicina</span> Molecular João Lobo Antunes,
Av. Prof. <span class=SpellE>Egas</span> Moniz, 1649-028 <span class=SpellE>Lisboa</span><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Carbon
Monoxide is endogenously produced in humans and has several physiological roles
such as signalling, vasodilatory, anti-apoptotic and anti-inflammatory <span
class=GramE>properties[</span>1]. Carbon monoxide releasing molecules (CORMs)
are designed to release CO in a controlled and directed manner, which allows
avoiding blood poisoning and thus constituting a safe alternative to
utilization of CO gas.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>CORMs also
have antimicrobial properties with capacity to kill human pathogens, such as
Escherichia coli, Helicobacter pylori, Pseudomonas aeruginosa and Plasmodium <span
class=GramE>species[</span>2–6]. However, so far, the amounts required are too
high to be usable. Therefore, we produced conjugated CORMs that combine the
CORM molecule with a known <span class=GramE>antibiotic, and</span> analysed
its antimicrobial properties. We observed that these compounds are more
effective against E. coli, S. aureus and Plasmodium species. The intracellular
release of CO was demonstrated by using a turn-on fluorescent probe COP-1.
Insights on the mode of action of these conjugated CORMs will be presented.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Camila Mariano</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Cell Biology, Centrosome-Cilia; Viruses<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Manipulation
of the Centrosome-Cilia complex by Viruses<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Camila
Mariano, Mónica Bettencourt-<span class=GramE>Dias</span> and Maria João
Amorim.<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência – Cell Cycle Regulation &amp; Cell Biology of Viral
Infection Labs.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>As obligate
intracellular parasites, viruses manipulate the host machinery to sustain their
life cycles. Interactions between viruses and the host cell cytoskeleton,
particularly with microtubules, are widely established, whereas interactions
with the centrosome-cilia complex have been profoundly overlooked. Centrosomes
and cilia are essential for microtubule organisation and signalling, but
questions remain regarding their structure/function regulation. Interestingly,
studies indicate that some viruses can manipulate the centrosome-cilia complex,
altering its structure and/or function, thus, opening new questions in cell
biology and virology. Therefore, I will initially investigate a panel of
viruses that alter the centrosome-cilia complex, selecting those with the
strongest phenotype. I will then proceed to research the molecular mechanism
underlying the observed phenotype, as well as its physiological relevance. With
this project, I expect to obtain insights into the regulation of the centrosome-cilia
complex and uncover new viral strategies to manipulate the host, which can
unveil new targets for antiviral therapies.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Carina <span class=SpellE>Andreia</span> Ribeiro <span
class=SpellE>Galhofa</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Microbiology, Quorum sensing<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Characterization
of the DPO quorum sensing system in Bacteroides <span class=SpellE>thetaiotaomicron</span><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Carina <span
class=SpellE>Galhofa</span> and Karina Xavier<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><i><span lang=EN-GB style='mso-ansi-language:EN-GB'>Bacteroides
<span class=SpellE>thetaiotaomicron</span> (B. theta)</span></i><span
lang=EN-GB style='mso-ansi-language:EN-GB'> is a prevalent member of the human
gut microbiota with important functions in host nutrition and immune system maturation.
One reason for its success in this competitive environment, which is home for
hundreds of different bacterial species, is due to its remarkable capacity to
degrade and utilize dietary and host polysaccharides. Production of saccharolytic
enzymes is among the several collective behaviours orchestrated by quorum
sensing (QS) systems. QS depends on the production and detection of signalling
molecules called autoinducers that accumulate in the environment in function of
cell density. In <i>B. theta</i>, the production of autoinducers hasn’t been
reported so far.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>In the
present study, culture supernatants of <i>B. theta</i> were incubated with a
Vibrio cholerae based bacterial QS reporter system to detect the DPO
autoinducer signal. This bioassay provided evidence for the presence of DPO
activity in <i>B. theta</i> cultures. Subsequentially, the same assay was used
to show that other Bacteroides species are potential producers of DPO. Genome
analysis of <i>B. theta</i> suggested the presence of three potential synthases
for DPO. The characterization of the DPO QS system in <i>B. theta</i> is
underway to better understand the molecular mechanisms regulating group
behaviours in Bacteroides and their role in the context of the gut microbiota
community. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Catarina Isabel <span class=SpellE>Brandão</span> <span
class=SpellE>Candeias</span> </span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Microbiology<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>A targeted
approach to control Streptococcus pneumoniae infection with minimum impact on
the microbiota <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Catarina <span
class=SpellE>Candeias</span> (1), Maria João Amorim (2), Raquel Sá-<span
class=SpellE>Leão</span> (1)<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(<span
class=GramE>1)Laboratory</span> of Molecular Microbiology of Human Pathogens,
Instituto de <span class=SpellE>Tecnologia</span> <span class=SpellE>Química</span>
e <span class=SpellE>Biológica</span> António Xavier, <span class=SpellE>Universidade</span>
Nova de <span class=SpellE>Lisboa</span> (ITQB-NOVA), Oeiras, Portugal;<br>
(2) Cell Biology of Viral Infection Laboratory, IGC, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Streptococcus
pneumoniae is a leading cause of morbidity and mortality worldwide. It is
frequently associated with respiratory viruses, such as influenza virus,
leading to secondary bacterial pneumonia, a common cause of hospitalization and
death. S. pneumoniae asymptomatically colonizes the nasopharynx of healthy
people. Colonization always precedes infection and is necessary for
transmission. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>This
project aims to identify human commensal strains that can act as <span
class=SpellE>biotherapeuticals</span> to contain pneumococcal colonization, <span
class=GramE>as a means to</span> prevent infection, particularly secondary
pneumococcal pneumonia after influenza A virus infection.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Previously,
an in vitro screen identified seven candidate commensal streptococci strains
with antimicrobial activity against S. pneumoniae. To understand if the
inhibition is maintained in more complex settings, in vitro and in vivo
experiments were planned. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Mice
colonization assays to evaluate colonization ability of both commensal and
pneumococcal strains were performed. In our experimental model (female 8 weeks
old C57BL/6 mice inoculated intranasally with 107 <span class=SpellE>cfu</span>
in 10 <span class=SpellE>μL</span> and sacrificed after 3 days) all
pneumococcal strains and two candidate strains (B and F) were able to colonize
mice. Bacterial loads recovered from nasal wash samples of mice inoculated with
pneumococcal strains (104-105 <span class=SpellE>cfu</span>/mL) were, however,
higher than the ones obtained for candidate strains (102-103 <span
class=SpellE>cfu</span>/mL). Attempts to increase bacterial load of strains B
and F were carried out with no significant success. Long-term colonization
assays were also performed, and except for candidate strain B, all strains can
colonize mice for at least 21 days.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>In vivo
competition assays for three pneumococcal strains and candidate strains B and F
were performed in two experimental designs: 3-day lagged and simultaneous
inoculations. These results are still under analysis and will be presented at
the meeting.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Catarina Pedro</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Mitosis; Transcription Inactivation; Cell Identity <o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>How
mitotic transcription inactivation can modulate cell identity and cell fate<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Catarina
Pedro, <span class=SpellE>Arunabha</span> Bose and Raquel Oliveira<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Transcriptional
events are overall aborted upon mitotic entry, in a process described as
mitotic transcription inactivation. During this process, most of transcription
regulators that control gene expression are evicted from the chromatin. Yet,
the reason for this transient shutdown remains unclear. Moreover, contrarily to
what was initially thought, some regions of the mitotic chromatin are still
transcribed throughout mitosis, implying that mitotic transcription
inactivation (MTI) is spatially regulated. This project aims to uncover how MTI
impacts on cell development. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Stem cells
are a unique model system to study the consequences of MTI given how the
mitotic transcriptional shutdown has been pinpointed as a possible time window
for the cell to either maintain and propagate a specific transcriptional
program to the daughter cells, ensuring cell identity, or to perform a
transcriptional reset and activate new transcriptional and differentiation
patterns.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>We will
take advantage of possible MTI regulators identified recently, and additional
ones uncovered by the host lab, to induce their acute and time-controlled
inactivation upon mitotic entry. Developed strategies for MTI impairment will
then be used to understand how this process determines transcriptional programs
and their propagation throughout cell division, defining cell fate decisions
and cell identity.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Daniela Cristina de Henriques <span class=SpellE>Brás</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Cell biology; Viral Infection; Mitochondria<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Mitochondrial
related processes that protect the host against IAV infection<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Daniela <span
class=SpellE>Brás</span> and Maria <span class=SpellE>Amourim</span><o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Viruses
adapt to the host they infect, to facilitate viral replication, escape immune
detection, and perpetuate viral transmission. Recently, it was shown that cells
use <span class=SpellE>membraneless</span> organelles to organize biological
reactions that arise by liquid-liquid phase separation (LLPS). During phase
separation, particles in a homogenous solution spontaneously separate and
become insoluble from the milieu establishing a boundary that is permeable only
to few selected particles. How viruses take advantage of phase separation to
facilitate their viral lifecycle is still unclear.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>We found
that influenza A viruses (IAV), known as one of the leading causes of severe
respiratory illness in the world, form liquid inclusions that may arise by LLPS
and that develop close to mitochondria and at ERES. Interestingly, many viral
lifecycles were shown to form specialized platforms, called viral inclusions,
that are essential for viral genome replication and/or for the assembly of
progeny neo-virions, that also attract mitochondria. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Using whole
proteome mass spectrometry-based solubility assays, we found that during IAV
infection, 2 viral factors, NS1 and progeny RNA, and 412 cellular factors, many
of them related with mitochondrial and ER metabolism, undergo phase
transitions. We found that changes in the solubility of mitochondrial proteins
were not observed when infected cells are treated to block liquid inclusion
formation. These results suggest that changes in the solubility of
mitochondrial proteins are somehow related with the formation and/or
maintenance of viral inclusions during infection. Supporting our hypothesis
that IAV infection induces phase transitions of key metabolic factors to
sustain and/or promote viral infection, pharmacological inhibition of the
electron respiratory transport chain dissolves IAV liquid inclusions and
abrogates infection. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>To
investigate, whether at an organismal level, IAV liquid inclusions are
physiological relevant, mice were infected and treated with <span class=SpellE>nucleozin</span>
(NCZ), a drug that hardens IAV liquid inclusions. Immunofluorescence images of
lungs of infected mice show that NCZ treatment also hardens liquid inclusions
in vivo. These findings show that it is possible to manipulate the properties
of viral inclusions in a mouse model of IAV infection, suggesting that LLPS
occurs in vivo and that interfering with its material properties may impact
disease outcome.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Diana Reis-<span class=SpellE>Barata</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Plant biology<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>The
interplay between TPS proteins and the SnRK1 kinase in the regulation of plant
growth and development<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Diana Reis-<span
class=SpellE>Barata</span> (1,2,3), John Lunn (2), Mark Stitt (2), Elena <span
class=SpellE>Baena</span>-Gonzalez (1)<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto Gulbenkian de Ciência; Rua da Quinta Grande 6, 2780-156 Oeiras,
Portugal <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
Max Planck Institute of Molecular Plant Physiology, Am <span class=SpellE>Mühlenberg</span>
1, 14476 Potsdam <span class=SpellE>Golm</span>, Germany<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(3)
Instituto de <span class=SpellE>Tecnologia</span> <span class=SpellE>Química</span>
e <span class=SpellE>Biológica</span>, <span class=SpellE>Universidade</span>
Nova de <span class=SpellE>Lisboa</span>, Av. Da <span class=SpellE>República</span>,
2780-157 Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Trehalose-6-phosphate
(T6P) is a potent signal metabolite required for maintaining plant sucrose
homeostasis and for adjusting plant growth and development to the available
carbon resources. It is synthesized from UDP-glucose (UDPG) and
glucose-6-phosphate (G6P) by the action of the T6P synthase TPS1. In actively
growing tissues, T6P acts partly by repressing the SNF1-related protein kinase1
(SnRK1) via still unknown mechanisms.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>The SnRK1
kinase is crucial for adjusting cellular metabolism and growth in situations of
energy deficit, thereby promoting tolerance to a wide variety of stress
conditions. In addition, it affects virtually all aspects of plant growth and
development, from seed germination to senescence, being considered a major
factor underlying plant developmental plasticity. SnRK1 activity is largely
regulated by carbon availability, being activated in response to carbon
starvation and inhibited by sugars, such as T6P.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Several
lines of evidence suggest that specific class II TPS proteins may act as
intermediary factors linking T6P and SnRK1. We are currently investigating the
impact of specific TPSs on SnRK1 <span class=SpellE>signaling</span> and the
underlying mechanisms.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span class=SpellE><span lang=EN-GB>Diogo</span></span><span lang=EN-GB>
Pinto Pereira de <span class=SpellE>Athayde</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Structural Biology, Zika Virus<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Engineering
effective diagnostics tools for Zika virus infections: Biophysical and
structural characterization of engineering proteins against Zika Virus Envelope
(E) antigen</span><span lang=EN-GB style='color:windowtext;mso-ansi-language:
EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span class=SpellE><span lang=EN-GB style='mso-ansi-language:
EN-GB'>Diogo</span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'> <span
class=SpellE>Athayde</span> (1), Isabelle Viana (2), Roberto <span
class=SpellE>Lins</span> (2), Maria João Amorim (3<span class=GramE>), <span
style='mso-spacerun:yes'> </span>Margarida</span> Archer (1)</span><span
lang=EN-GB style='color:windowtext;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
ITQB-NOVA, <span class=SpellE>Lisboa</span>, Portugal<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
FIOCRUZ, Recife, <span class=SpellE>Brasil</span><o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(3)
Instituto Gulbenkian de Ciência, <span class=SpellE>Lisboa</span>, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Zika virus
(ZIKV) is an <span class=GramE>arthropod-borne</span> Flavivirus, which
includes other important human pathogens such as Dengue (DENV), West Nile (WNV)
and Yellow Fever (YFV) viruses. In 2015, an epidemic outbreak with unforeseen
proportions occurred in Brazil with 1,300,000 suspected cases. Human infection with
ZIKV can cause mild febrile flu-like symptoms to severe neurologic disorders,
such as microcephaly and Guillain-Barre syndrome. Major efforts have been made
to develop reliable serological diagnostics for ZIKV as well as effective
treatments and vaccines, however, the lack of information on the structural
determinants underlying viral specificity and host interactions <span
class=GramE>still remains</span> a challenge. Our collaborators at <span
class=SpellE>Fiocruz</span>, Recife, Brazil, computationally engineered small
proteins that mimic neutralizing antibodies against ZIKV with high binding
affinity. This was achieved by grafting the binding motif of the antibodies to
an already structurally characterized <span class=GramE>E.coli</span> protein.
Through a series of point mutations, validated by molecular dynamics simulations,
a theoretically stable chimeric protein was obtained. My goal is to structural
and biophysically characterize several of these chimeric proteins and complexes
between them and the respective E or NS1 viral proteins. The work herein
presented describes the structural determination for one of these engineered
proteins through X-ray crystallography and its biophysical characterization by
circular dichroism (CD) and thermostability assays (TSA). This knowledge may
help the development of diagnostics tools and prophylactic vaccines against
ZIKV.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Francisco Ferreira da Silva <span class=SpellE>Paupério</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Metabolism; Evolution; Bioinformatics<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>A
computational framework for the evolution of metabolic networks in the gut
environment<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Francisco <span
class=SpellE>Paupério</span>, Isabel <span class=SpellE>Gordo</span> &amp;
Tiago Paixão<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Evolutionary
Biology group and The Quantitative Biology &amp; Digital Science unit,
Instituto Gulbenkian de Ciência<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>The
mammalian gut hosts a complex community, composed of many interacting microbial
species. This system interacts with peripheral organs providing important
homeostasis functions by modulating the metabolic composition of the gut.
However, little is known about evolutionary and ecological processes in the
gut. Therefore, it is crucial to understand its community composition, its
evolution and interplay with the gut environment (<span class=SpellE>eg</span>:
diet). Motivated by recent results that suggest that E. coli evolution in the
mouse gut is shaped by the metabolites present in this ecosystem environment,
we explore in vitro environments composed by different sugar sources to explore
the effect of single gene knockouts on the fitness of <i>E. coli</i>. To study
these effects, we use a genome-scale metabolic reconstructions (GEMs), which
were previously found to be very accurate at predicting growth rate (fitness)
of <i>E. coli</i>. Using this model, we predicted the flows inside <i>E. coli</i>
metabolic pathways and growth rates in different environments. We explored the
implications of mutations in the metabolic model of <i>E. coli</i>. We found
that even if we change the sugar source in the environment, the distribution of
fitness effects (DFE) of every single mutant remains the same across
environments. We found two main peaks: one around the wild <span class=GramE>type</span>
fitness, and another one of lethal mutations with fitness close to 0. They all
present an intermediate peak around 0.5. This points towards that adaptation
may require a modest number of genes when they have a small effect or few genes
when their effect is large. Furthermore, when we increase the number of
mutations, we also see similar DFE in multiple environments. The work here
demonstrates the power of genome-scale metabolic reconstruction models to study
adaptation of microbes and genome evolution.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Joana <span class=SpellE>Estudante</span> Saraiva</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Tissue Mechanics; Histone Modifications; Collective Cell Migration<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Mechanical
fine-tuning of chromatin accessibility triggers neural crest collective cell
migration<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Joana
Saraiva and Elías H Barriga<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='background:white;mso-ansi-language:
EN-GB'>Mechanisms of Morphogenesis Lab, Gulbenkian Institute of Science (IGC),
2780-156 Oeiras, Portugal</span><span lang=EN-GB style='font-family:"Times New Roman",serif;
mso-bidi-font-family:Calibri;mso-bidi-theme-font:minor-latin;color:windowtext;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Temporal
and spatial coordination of the onset of collective cell migration (CCM) is
essential for morphogenesis in physiology and disease. Recent findings show
that mechanical cues from the environment modify cell behaviour and gene
expression to trigger CCM in vivo. For instances, the CCM of neural crest cells
is triggered by stiffening of their substrate, the mesoderm. Yet, the mechanism
underlying this molecular response to mechanics remains unclear. In vitro
studies suggest that one of the effects of mechanical signals is chromatin
modification. Here by combining cutting-edge mechanical, <span class=GramE>imaging</span>
and molecular tools, we show that mechanical cues from the substrate fine-tune
chromatin accessibility by modulating the occurrence of a recently discovered
chromatin modification, named histone <span class=SpellE>serotonylation</span>.
Furthermore, we show that this <span class=SpellE>serotonylation</span> supports
epithelial to mesenchymal transition in response to mesoderm stiffening.
Overall, our results contribute with in vivo insights to the growing field of
‘mechano-epigenetics’.<o:p></o:p></span></p>

<p class=Normal1><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Joana Sofia Monteiro Batista</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Campylobacter <span class=SpellE>jejuni</span>, Haem<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>The haem
biosynthesis pathway of Campylobacter <span class=SpellE>jejuni</span> <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Joana
Batista, Jordi <span class=SpellE>Zamarreño</span> Beas, Marco A. M. <span
class=SpellE>Videira</span>, and <span class=SpellE>Lígia</span> M. Saraiva<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
de <span class=SpellE>Tecnologia</span> <span class=SpellE>Química</span> e <span
class=SpellE>Biológica</span> António Xavier, <span class=SpellE>Universidade</span>
Nova de <span class=SpellE>Lisboa</span>, Avenida da <span class=SpellE>República</span>
(EAN), 2780-157 Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Campylobacter
<span class=SpellE>jejuni</span> is a Gram-negative bacterium and the most
common cause of gastroenteritis worldwide. This pathogen is responsible of
approximately 1/4 cases of diarrheal disease, which may be fatal in people with
other comorbidities. <br>
Haem is an iron-protoporphyrin molecule that is essential to all living
organisms. As such, this cofactor is required for survival of most pathogens
within the infecting host, and several pathogens have dedicated haem <span
class=SpellE>biosyntesis</span> pathways. Our research focus on elucidating how
C. <span class=SpellE>jejuni</span> produces haem intracellularly. Our data
shows that C. <span class=SpellE>jejuni</span> uses the so-called
protoporphyrin-dependent pathway (PPD)3, in which two new proteins are active
in this microorganism. We also uncovered important protein-protein interactions
involving these novel proteins.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Therefore,
this work sheds novel insights into Campylobacter <span class=SpellE>jejuni</span>
haem biosynthesis, unravelling potential targets to fight infection. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span class=SpellE><span lang=EN-GB>Kátia</span></span><span lang=EN-GB>
Ribeiro de Jesus</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Immunology, Infection, Physiology<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>To Stress…To
Resist? The Power <span class=GramE>Of</span> Stress Responses In Sepsis</span><span
lang=EN-GB style='font-size:10.0pt;font-family:"Arial",sans-serif;mso-ansi-language:
EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Katia Jesus
(1), André Barros (1), Elsa <span class=SpellE>Seixas</span> (1), Luís <span
class=SpellE>Moita</span> (1)(2)</span><span lang=EN-GB style='color:windowtext;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)&nbsp;Instituto
Gulbenkian de Ciência, Rua da Quinta Grande 6, 2780-156 Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
Instituto de <span class=SpellE>Histologia</span> e <span class=SpellE>Biologia</span>
do <span class=SpellE>Desenvolvimento</span>, <span class=SpellE>Faculdade</span>
de <span class=SpellE>Medicina</span>, <span class=SpellE>Universidade</span>&nbsp;de
<span class=SpellE>Lisboa</span>, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Restoration
of organismal homeostasis after an insult is the result of a balanced response
by the two main arms of host defence: disease tolerance and resistance. Inducing
stress responses can trigger disease tolerance mechanisms, as demonstrated in
several recent studies. There is growing evidence supporting the hypothesis
that this could be an effective strategy, much needed for treating complex
multifactorial conditions, such as Sepsis. However, the protective mechanisms
initiated by stress responses have not been systematically studied. Thus, it is
of key importance to understand the impact of stress responses on resistance
mechanisms and the role of the immune system in these potential therapeutic
approaches.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Our
laboratory has recently published evidence that inducing mild disruptions in
the mitochondria can trigger disease tolerance programs. Herein, a murine model
of Severe Sepsis showed increased survival and less tissue damage when treated
with mitochondria-stress inducing drugs. Currently, we are working towards
dissecting the impact of this stress response in resistance programs. So far,
we have gathered compelling evidence that inducing mild disruption in the
mitochondria is also triggering pathogen clearance, a hallmark of resistance
responses. Furthermore, we have supporting data showing that immune cell
populations and their profile are impacted by such treatment and could be core
players on the underlying protective mechanisms being triggered. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Understanding
the mechanistic bases by which stress responses can induce both disease
tolerance and resistance and dissecting the fine-tuned balanced cross talk that
helps restore homeostasis could contribute to the development of synergistic
therapeutic approaches to life threatening conditions, including sepsis.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span class=SpellE><span lang=EN-GB>Khira</span></span><span lang=EN-GB>
Amara</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Microbiology, Development, Spore<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>A
hierarchical cascade of protein-protein cross-linking governs assembly of the
Bacillus subtilis inner coat hub <span class=SpellE>SafA</span><o:p></o:p></span></p>

<p class=MsoNormal><span class=SpellE><span lang=EN-GB style='mso-ansi-language:
EN-GB'>Khira</span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'>
Amara, Bruno Fernandes, Tiago <span class=SpellE>Pais</span>, Catarina
Fernandes, Mónica <span class=GramE>Serrano</span> and Adriano O. Henriques<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
De <span class=SpellE>Tecnologia</span> <span class=SpellE>Química</span> E <span
class=SpellE>Biológica</span>, <span class=SpellE>Universidade</span> Nova De <span
class=SpellE>Lisboa</span>, Avenida Da <span class=SpellE>República</span>, <span
class=SpellE>Apartado</span> 127, 2781-901 Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>During
Bacillus subtilis spore’s development, more than 80 proteins are assembled
around the spore to form a protective coat which is differentiated into an
inner and an <span class=GramE>outer substructures</span>. <span class=SpellE>SafA</span>
is the hub protein for assembly of the inner coat. One form of <span
class=SpellE>SafA</span>, called C30, which is generated by internal
translation, recruits a transglutaminase, <span class=SpellE>Tgl</span>, to the
spore surface. <span class=SpellE>Tgl</span> cross-links its substrates through
ε-(γ-<span class=GramE>glutamyl)<span class=SpellE>lysil</span></span> bonds in
following assembly, it cross-links C30. Since C30 forms hexamers in vitro
thought to contain two <span class=SpellE>disulfide</span> bonds, we tested
whether <span class=SpellE>disulfide</span> formation could be a requirement
for <span class=SpellE>Tgl</span> crosslinking, which requires a glutamyl (Q)
donor and a <span class=SpellE>lysil</span> (K) acceptor. To test this
possibility, versions of C30 lacking <span class=SpellE>Cys</span> (C to S),
lysine (K to A), or both amino acids (S and K to A) were generated and tested
for self-assembly and <span class=SpellE>Tgl</span> cross-linking, in vitro. We
show that <span class=SpellE>disulfide</span> bond formation is important for
the kinetics of cross-linking of C30 by <span class=SpellE>Tgl</span> and
prevents an activity of the enzyme that most likely leads to the
circularization of C30. Moreover, <span class=SpellE>disulfide</span> bond
formation is crucial for the crosslinking of C30 into species of even higher
order. We propose a model in which the cross-linking of C30 involves a hierarchical
cascade of covalent protein cross-linking reactions. In this model, <span
class=SpellE>disulfide</span> bond formation stabilizes a C30 hexamer and
orients Q and K residues for the efficient cross-linking by <span class=SpellE>Tgl</span>.
In turn, <span class=SpellE>Tgl</span> exerts a spot-welding activity
fortifying a pre-formed, <span class=SpellE>disulfide</span> cross-linked C30
hexamer, while also leading to the cross-linking of higher order complexes.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Lucrezia C. <span class=SpellE>Ferme</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
<span class=SpellE>pseudostratification</span>, neuroepithelia, tissue packing<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Investigating
the influence of <span class=SpellE>pseudostratification</span> for
neuroepithelial development<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Lucrezia <span
class=SpellE>Ferme</span> and Caren Norden<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Pseudostratified
epithelia (PSE) are highly proliferative tissues that serve as organ precursors
in many developmental contexts in invertebrates and vertebrates. This
prevalence suggests an advantage of this tissue arrangement for correct
organogenesis, but what exactly this advantage could be is currently unclear.
One possibility is that <span class=SpellE>pseudostratification</span> allows
for more efficient packing of cells within a monolayered tissue. We speculate
this because PSE feature elongated epithelial cells whose nuclei locate at
different depths along the apicobasal axis, resulting in dense tissue packing.
This in turn could facilitate correct organogenesis.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>To
investigate these ideas, I use the zebrafish retina as a model of
pseudostratified neuroepithelia because of the fast development of the embryos
and accessibility of the tissue. I imaged the developing retina over time,
extracted <span class=GramE>single-cell</span> features and followed their
changes. Preliminary results revealed that the retinal PSE gets close to
theoretical maximum packing densities just before neuronal delamination. While
we would expect the tissue to transit from a flowing to a rigid state, we
observe that nuclear migration and cell movements are not impaired even at
these packing levels.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>To explore
how proliferating PSE can reach dense packing while retaining a seemingly
fluid-like behaviour, I aim to measure the mechanical stresses generated within
the retinal neuroepithelium and build a theoretical model of nuclear packing
that will enable a deeper appreciation of possible tissue-scale effects of <span
class=SpellE>pseudostrafication</span>. This will allow to understand the
relevance of <span class=SpellE>pseudostratification</span> for neuroepithelial
development and decipher how ever-changing tissues coordinate proliferation and
differentiation.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Maria Virginia Ramirez Montoya</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Microbiota, Dysbiosis, Quorum-Sensing<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>The impact
of quorum sensing manipulation during recovery from diet-induced dysbiosis.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Maria
Montoya, Jocelyne <span class=SpellE>Demengeot</span> and Karina Xavier</span><span
lang=EN-GB style='color:windowtext;mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Rua da Quinta Grande 6, 2780-156 Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Bacteria
are characterized by their ability to act collectively thanks to a chemical
communication process called quorum sensing (QS). This system comprises the
control of small <span class=SpellE>signaling</span> molecules called
autoinducers that allow the coordination of collective <span class=SpellE>behaviors</span>.
Although several of these molecules are species-specific, others are important
in interspecies communication and are likely to be relevant in complex
microbial communities such as those found in the gut. The intestinal microbiota
comprises trillions of microorganisms that, besides interacting with each
other, live in intimate association with their host, where they are involved in
multiple fundamental processes. Since microbe-microbe and microbe-host
interactions promote host homeostasis, compositional and functional alterations
of the microbiota (dysbiosis) have been observed in multiple diseases and
syndromes. In this context, diet represents one of the main environmental
disturbances. In this research, we will study the interactions between
microorganisms in recovery from diet-induced dysbiosis in the presence and
absence of the quorum-sensing signal autoinducer-2 (AI-2) and how these
interactions affect and are affected by the host. For that purpose, we will use
a gnotobiotic mice model colonized with a defined microbiota to identify
genetic and functional traits responding to AI-2. On the side of the
microbiota, we will apply a multi-omics approach covering metagenomics,
transcriptomics, and metabolomics characterization. With this strategy, we will
identify molecular mechanisms by which AI-2 manipulates the microbe community.
Finally, we will evaluate the immunological and physiological impact of AI-2
interventions on the host side. By integrating our findings, we hope to
decipher the mechanisms regulating diet-microbiota-host interactions so that
causality can be established. Preliminary results show that changes in diet
(from a standard to a high-fat high-sugar diet) in the context of a complex
microbiota community, affect different host parameters such as body and organs
weight and fat deposition. These and other results will be used as a reference
for the effects of this dietary regimen on the microbiota and the host and will
allow us to investigate the recapitulation power of the gnotobiotic model. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Mariana Maia Gil</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Development; Imaging; Cell Biology<o:p></o:p></span></p>

<p class=MsoNormal><i><span lang=EN-GB style='font-size:16.0pt;mso-bidi-font-size:
12.0pt;font-family:"Calibri Light",sans-serif;mso-ascii-theme-font:major-latin;
mso-fareast-font-family:Calibri;mso-fareast-theme-font:minor-latin;mso-hansi-theme-font:
major-latin;mso-bidi-theme-font:major-latin;mso-ansi-language:EN-GB'>Investigating
the influence of nuclear and tissue properties in pseudostratified neuroepithelial
development<o:p></o:p></span></i></p>

<p class=MsoNormal><u><span lang=EN-GB style='mso-ansi-language:EN-GB'>Mariana
Maia Gil</span></u><sup><span lang=EN-GB style='mso-ansi-language:EN-GB'>1</span></sup><span
lang=EN-GB style='mso-ansi-language:EN-GB'>, Roman Belousov<sup>2</sup>, Anna
Erzberger<sup>2</sup> and Caren Norden<sup>1<o:p></o:p></sup></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-fareast-theme-font:minor-fareast;mso-bidi-font-family:
Calibri;mso-bidi-theme-font:minor-latin;color:#5A5A5A;mso-themecolor:text1;
mso-themetint:165;letter-spacing:.75pt;mso-ansi-language:EN-GB'>(1) Instituto
Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-fareast-theme-font:minor-fareast;mso-bidi-font-family:
Calibri;mso-bidi-theme-font:minor-latin;color:#5A5A5A;mso-themecolor:text1;
mso-themetint:165;letter-spacing:.75pt;mso-ansi-language:EN-GB'>(2) EMBL,
Heidelberg, Germany<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Pseudostratified
epithelia (PSE) are highly proliferative tissues that serve as organ precursors
in diverse developmental contexts. In all PSE, nuclei need to move to the
apical side before mitosis, which enables successful tissue maturation. This
phenomenon is called apical nuclear migration and is mainly driven by the
actomyosin cytoskeleton. However, during PSE development nuclear packing
increases. It is still not understood how nuclei can still move despite the
increasingly crowded environment. Here we use the zebrafish retina to
investigate whether nuclear properties themselves could facilitate nuclear movements
in an increasingly packed PSE.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>In vitro
studies show that cells deficient of Lamin A/C, a nuclear lamina protein, are
more deformable and migrate more efficiently through physical constrains of the
environment. It was recently found that nuclei in zebrafish neuroepithelia
express low levels of Lamin A/C and are also highly deformable, as would be
expected under these circumstances. However, whether and how this observed
deformability facilitates nuclear movements through crowded PSE, is not yet
clear.&nbsp;<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>To close
these gaps, we genetically overexpressed Lamin A/C and studied nuclei movements
<i>in vivo</i>. Here we show that nuclei overexpressing Lamin A/C take longer
to move apically and often start already closer to the apical side.
Furthermore, cell-transplantations showed that in an environment where all
nuclei overexpress Lamin A/C apical nuclear migration of control nuclei is
hampered. This suggest that Lamin A/C overexpression, either on single-nuclei
or in all nuclei across the tissue, affect apical nuclear migration. Currently,
we are using nuclei positions over time to infer nuclear properties.
Preliminary data suggests that nuclei overexpressing Lamin A/C are stiffer than
control nuclei. Overall, our results gather new insights into how nuclear
properties influence neuroepithelial development and disclose the relevance of
nuclear deformability for tissue maturation.<o:p></o:p></span></p>

<p class=Normal1><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Mariana Ribeiro Natalino</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Evolution<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Role of
nutrient-sensing during the evolutionary adaptation to DNA replication stress<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Mariana
Natalino and Marco Fumasoni</span><span lang=EN-GB style='color:windowtext;
mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoSubtitle><i><span lang=EN-GB style='mso-ansi-language:EN-GB'>Genome
Maintenance and Evolution Lab</span></i><span lang=EN-GB style='mso-ansi-language:
EN-GB'> - Instituto Gulbenkian de Ciência, Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>How
nutrient availability influences the evolutionary adaptation to an
intracellular selective pressure remains largely unknown. The target of rapamycin
(TOR) is a conserved pathway responsible for regulating multiple cellular
processes in response to nutrient availability. Recent evidence <span
class=GramE>have</span> shown that the TOR pathway plays a role in the
maintenance of genome stability and the acute response to replication stress.
This project aims to investigate how nutrient perception affects the
evolutionary adaptation to perturbations impairing genome replication and
reducing cellular fitness. I will take advantage of experimental evolution with
S. cerevisiae combined with targeted mutations in the TOR pathway to identify
the adaptive mutations allowing cells to recover <span class=GramE>fitness, and</span>
study the mechanisms by which nutrient-sensing impacts evolutionary adaptation.
The results will be relevant for understanding the role of nutrient
availability in evolution in nature and in cancer progression, where cells
experiencing replication stress are recurrently exposed to novel nutrient
niches.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Miguel Correia Landum</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Antiviral Innate Immunity, Drosophila, Toll Pathway<o:p></o:p></span></p>

<p class=RYHEADING2><span class=HeaderChar><span lang=EN-GB style='mso-ansi-language:
EN-GB'>Non-canonical activation of the Drosophila Toll pathway by viral
infection</span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Nelson
Martins (1), Álvaro Gil Ferreira (3<span class=GramE>),Marc</span> <span
class=SpellE>Veldhoen</span> (4), Luís Teixeira (1,2)<o:p></o:p></span></p>

<p class=MsoNormal style='margin:0cm;mso-add-space:auto'><span lang=EN-GB
style='font-size:10.0pt;mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:
minor-fareast;mso-bidi-font-family:Calibri;mso-bidi-theme-font:minor-latin;
color:#5A5A5A;mso-themecolor:text1;mso-themetint:165;letter-spacing:.75pt;
mso-ansi-language:EN-GB'>(1) Instituto Gulbenkian de <span class=SpellE>Ciências</span>,
Oeiras, Portugal<o:p></o:p></span></p>

<p class=MsoNormal style='margin:0cm;mso-add-space:auto'><span lang=EN-GB
style='font-size:10.0pt;mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:
minor-fareast;color:#5A5A5A;mso-themecolor:text1;mso-themetint:165;letter-spacing:
.75pt;mso-ansi-language:EN-GB'>(2) <span class=SpellE>Faculdade</span></span><span
lang=EN-GB style='font-size:10.0pt;mso-fareast-font-family:"Times New Roman";
mso-fareast-theme-font:minor-fareast;mso-bidi-font-family:Calibri;mso-bidi-theme-font:
minor-latin;color:#5A5A5A;mso-themecolor:text1;mso-themetint:165;letter-spacing:
.75pt;mso-ansi-language:EN-GB'> de <span class=SpellE>Medicina</span> da <span
class=SpellE>Universidade</span> de <span class=SpellE>Lisboa</span>, <span
class=SpellE>Lisboa</span>, Portugal<o:p></o:p></span></p>

<p class=MsoNormal style='margin:0cm;mso-add-space:auto'><span lang=EN-GB
style='font-size:10.0pt;mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:
minor-fareast;color:#5A5A5A;mso-themecolor:text1;mso-themetint:165;letter-spacing:
.75pt;mso-ansi-language:EN-GB'>(3) Grupo</span><span lang=EN-GB
style='font-size:10.0pt;mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:
minor-fareast;mso-bidi-font-family:Calibri;mso-bidi-theme-font:minor-latin;
color:#5A5A5A;mso-themecolor:text1;mso-themetint:165;letter-spacing:.75pt;
mso-ansi-language:EN-GB'> Mosquitos <span class=SpellE>Vetores</span>: <span
class=SpellE>Endossimbiontes</span> e <span class=SpellE>Interação</span> <span
class=SpellE>Patógeno-Vetor</span>, Instituto René <span class=SpellE>Rachou</span>
– <span class=SpellE>Fiocruz</span>, Belo Horizonte, MG, Brazil<o:p></o:p></span></p>

<p class=MsoNormal style='margin-bottom:0cm;margin-bottom:6.0pt;mso-margin-bottom-alt:
6.0pt;mso-margin-top-alt:6.0pt;mso-add-space:auto'><span lang=EN-GB
style='font-size:10.0pt;mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:
minor-fareast;color:#5A5A5A;mso-themecolor:text1;mso-themetint:165;letter-spacing:
.75pt;mso-ansi-language:EN-GB'>(</span><span lang=EN-GB style='font-size:10.0pt;
mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:minor-fareast;
mso-bidi-font-family:Calibri;mso-bidi-theme-font:minor-latin;color:#5A5A5A;
mso-themecolor:text1;mso-themetint:165;letter-spacing:.75pt;mso-ansi-language:
EN-GB'>4</span><span lang=EN-GB style='font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-fareast-theme-font:minor-fareast;color:#5A5A5A;
mso-themecolor:text1;mso-themetint:165;letter-spacing:.75pt;mso-ansi-language:
EN-GB'>) </span><span lang=EN-GB style='font-size:10.0pt;mso-fareast-font-family:
"Times New Roman";mso-fareast-theme-font:minor-fareast;mso-bidi-font-family:
Calibri;mso-bidi-theme-font:minor-latin;color:#5A5A5A;mso-themecolor:text1;
mso-themetint:165;letter-spacing:.75pt;mso-ansi-language:EN-GB'>Instituto de <span
class=SpellE>Medicina</span> Molecular João Lobo Antunes, <span class=SpellE>Lisboa</span>,
Portugal</span><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=Normal1><span lang=EN-GB style='mso-ansi-language:EN-GB'>The Toll
pathway was first identified to be important for dorsal-ventral patterning of
the Drosophila embryo, and later for immunity against bacteria and fungi. The
immune branch of the pathway is mainly required for <span class=SpellE>defense</span>
against pathogens through the activation of cellular immunity, and the
regulation of antimicrobial peptides (AMPs), and other <span class=GramE>small
secreted</span> peptides, like <span class=SpellE>Bomanins</span>. Although
Toll activation and activity against bacteria and fungi have been well
characterized, its importance against viruses was only recently shown in
Drosophila. Our work focuses on understanding the mechanism by which viral
infection activates the Drosophila Toll pathway and which target genes are
effectors of the antiviral defence. Our results show that none of the canonical
proteolytic enzymes described in Drosophila immunity and development signalling
is necessary for the activation upon viral infection. This may indicate a
virus-specific set of enzymes necessary for Toll activation. We did, however,
identify a new protease from the family of enzymes responsible for Toll
activation, as having an antiviral role. We are now testing if it is part of a
new pathway that activates Toll upon viral infection. Interestingly we showed
that this enzyme is important for <span class=SpellE>melanization</span>,
another innate immune response in flies. We are currently also exploring if <span
class=SpellE>melanization</span> or the crosstalk between the two pathways is
important for antiviral resistance. Finally, we are looking for the role of
AMPs and others known Toll effectors in antiviral immunity. <o:p></o:p></span></p>

<p class=Normal1><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Priscilla <span class=SpellE>Abena</span> <span
class=SpellE>Akyaw</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Innate Immunity, Drosophila Melanogaster, Disease Tolerance<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Identifying
mechanisms of disease tolerance to immunopathology in Drosophila melanogaster.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-fareast-font-family:"Times New Roman";
mso-fareast-theme-font:major-fareast;mso-ansi-language:EN-GB'>Priscilla</span><span
lang=EN-GB style='mso-ansi-language:EN-GB'>&nbsp;A. Akyaw<sup>1</sup>, Tânia F.
Paulo<sup>1</sup>, David Duneau<sup>2</sup>, <span class=SpellE>Élio</span>
Sucena<sup>1</sup><o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)&nbsp;Instituto
Gulbenkian de Ciência, Oeiras -Portugal<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)&nbsp;CNRS,
<span class=SpellE>Laboratoire</span> <span class=SpellE>Évolution</span> &amp;
<span class=SpellE>Diversité</span> <span class=SpellE>Biologique</span>, <span
class=SpellE>Université</span> Toulouse 3 Paul Sabatier, Toulouse, France<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Host
resistance to pathogens and disease tolerance of established infections are
known major components of immunity. When it comes to insect immunity, both
resistance and disease tolerance contribute significantly to host survival,
upon both systemic and oral infections. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Studies
into understanding host response to infection have targeted the mechanisms of
resistance, which is now well elucidated both in vertebrates and invertebrate
organisms, especially in Drosophila melanogaster. Strikingly, the same cannot
be said for disease tolerance mechanisms. Disease tolerance pertains to
mechanisms that favour the hosts ability to withstand and/or repair damage
inflicted either directly by the invading pathogen (i.e., virulence or
pathogen-associated injury) or damages caused indirectly by the host’s own
immune effectors during the process of resistance (i.e., immunopathology). In
both scenarios, very little is known about the mechanisms involved, their
genetic bases, and how they interact during and after an infection. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>We have
developed a protocol to characterize the response of the highly tractable
insect model D. melanogaster to immunopathology as quantified through survival
and reproduction traits after infection with the entomopathogen Pseudomonas <span
class=SpellE>entomophila</span>. We observe that both survival and reproductive
output are negatively affected confirming that part of the cost on survival and
reproduction elicited by bacterial infection is attributable to
immunopathology. Additionally, we show that this reduction in fitness traits is
also correlated with an activated immune response mainly characterized by the
upregulation of Anti-Microbial Peptides (AMP’s) and stress response pathways. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Using this
protocol, we will measure variation in disease tolerance across a large panel
of Isogenic Drosophila lines (DGRP), <span class=GramE>in order to</span>
perform a GWAS and identify candidate genes for this process. Ultimately, we
aim at characterizing the critical genetic players of the disease tolerance
mechanisms dealing with damage caused by immunopathology as well as infection
per se. This protocol may prove instrumental in identifying the impact of
disease tolerance on host fitness and how it relates to resistance mechanisms
in Drosophila melanogaster.<o:p></o:p></span></p>

<p class=Normal1><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Ravi Vishwakarma</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Habitat Loss <span class=GramE>And</span> Fragmentation, Species Traits,
Genetic Diversity<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Understanding
the genetic consequences of habitat loss, fragmentation across species through
spatial modelling<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Ravi Vishwakarma
(1), Gabriele Maria <span class=SpellE>Sgarlata</span> (1), Tiago <span
class=SpellE>Maie</span> (2), <span class=SpellE>Rémi</span> <span
class=SpellE>Tournebize</span> (1), <span class=SpellE>Lounès</span> <span
class=SpellE>Chikhi</span> (1,3)<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpFirst><span lang=EN-GB style='mso-ansi-language:EN-GB'>(1)
Instituto Gulbenkian de Ciência, Oeiras -Portugal <o:p></o:p></span></p>

<p class=MsoSubtitleCxSpMiddle><span lang=EN-GB style='mso-ansi-language:EN-GB'>(2)
Joint Research <span class=SpellE>Center</span> for Computational Biomedicine,
RWTH - Aachen University, Aachen, Germany<o:p></o:p></span></p>

<p class=MsoSubtitleCxSpLast><span lang=EN-GB style='mso-ansi-language:EN-GB'>(3)
<span class=SpellE>Laboratoire</span> <span class=SpellE>Évolution</span> &amp;
<span class=SpellE>Diversité</span> <span class=SpellE>Biologique</span> (EDB
UMR 5174), CNRS, IRD, UPS, <span class=SpellE>Université</span> de Toulouse
Midi-Pyrénées, Toulouse, France<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Habitat
Loss &amp; Fragmentation (HL&amp;F) along with global warming pose a serious
threat to world’s biodiversity by altering species habitats. Several studies on
animal species indicate a negative impact of HL&amp;F on genetic diversity.
Moreover, the adverse genetic effects vary widely across species. Despite
decades of work on the genetics of endangered species, we still lack knowledge
on the fine-scale consequences of HL&amp;F processes on genetic diversity. It
is important to improve our understanding on how species traits influence
patterns of genetic diversity before and during HL&amp;F processes. This is the
general goal of my PhD project. In this poster, I present our work on spatial
genetic diversity patterns under various scenarios of HL&amp;F while varying
some species traits using spatial simulations. More specifically, we used a
spatially explicit model to look at the pattern and temporal dynamics of
genetic diversity for various speeds of habitat range contraction that followed
an initial period of habitat expansions for species with different generation
times. Our preliminary analysis shows significant effects of both the contraction
speed and generation time on the dynamics and pattern of genetic diversity. For
instance, species with long generation time exhibit a different trajectory
wherein the genetic diversity can still increase during the initial phase of
range contractions, which is not the observed for species with short generation
time. These results are very preliminary since our hypothetical species only
differed in their generation time while keeping all other parameters identical.
Real species typically show correlation in different life history traits, <span
class=GramE>e.g.</span> longer generation times are typically associated with
lower population sizes, and growth rates. We are now working on finding
correlations in real biological data among different parameters to determine
whether more realistic scenarios will produce consistent or different results.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Romana <span class=SpellE>Yáñez</span></span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Plant Biology; Cell Biology; Microscopy<o:p></o:p></span></p>

<p class=RYHEADING2><span lang=EN-GB style='mso-ansi-language:EN-GB'>Investigating
A Role <span class=GramE>In</span> Translation For Plant SR Proteins Involved
In Stress Responses. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Romana
Yanez, Maria Hanulova, Paula Duque<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Oeiras -Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Several
cell processes enable living organisms to cope with environmental stress. In
plants, serine/arginine-rich (SR) proteins, which belong to a highly conserved family
of eukaryotic RNA-binding proteins, are major alternative splicing regulators
reported to modulate abiotic stress responses. Numerous SR proteins are known
to shuttle between the nucleus and the cytoplasm, with shuttling mammalian SR
proteins having been found to fulfil postsplicing roles, namely in mRNA export,
stability, and translation. Although several plant SR proteins have been
reported to exhibit nucleocytoplasmic shuttling, their postsplicing activities
remain to be investigated. This project aims at determining whether individual
Arabidopsis thaliana SR proteins involved in stress responses display
nucleocytoplasmic shuttling and play a direct role in translation. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>To
determine whether our selected SR proteins show nucleocytoplasmic shuttling, we
used the FLIP (Fluorescence Loss <span class=GramE>In</span> Photobleaching)
microscopy technique and transgenic A. thaliana lines expressing GFP-tagged SR
protein versions. The cytoplasm of A. thaliana root cells was continuously
bleached while measuring the fluorescence loss in the nucleus. Subsequently, a
potential SR protein nucleocytoplasmic shuttling activity was validated upon
treatment with leptomycin B, a nuclear export inhibitor. So far, we have
confirmed nucleocytoplasmic shuttling activity for two SR proteins and identified
another two shuttling members of the A. thaliana SR protein family. These
results suggest cytoplasmic functions in RNA metabolism for these splicing
factors. The effect of the shuttling SR proteins on translation will next be
investigated by ribosome fractionation and RNA-sequencing of the actively
translating mRNAs isolated from polysome fractions. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>In
establishing whether plant SR proteins also play broader roles apart from
splicing in gene expression regulation, this project will shed new light on the
functions of these important RNA-binding proteins in plant systems.<o:p></o:p></span></p>

<p class=MsoNormal align=left style='text-align:left'><span lang=EN-GB
style='mso-ansi-language:EN-GB;mso-no-proof:yes'><v:shape id="Picture_x0020_1"
 o:spid="_x0000_i1025" type="#_x0000_t75" alt="A picture containing graphical user interface&#10;&#10;Description automatically generated"
 style='width:273pt;height:149pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="AMeeGuS2022_BookOfAbstracts_20220128.fld/image009.png"
  o:title="A picture containing graphical user interface&#10;&#10;Description automatically generated"/>
</v:shape></span><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-GB>Victor Hugo de Mello Pessoa</span></h2>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Keywords:
Cell biology; Molecular Biology; Virology<o:p></o:p></span></p>

<p class=RYHEADING2><span class=SpellE><span lang=EN-GB style='mso-ansi-language:
EN-GB'>FLUorescent</span></span><span lang=EN-GB style='mso-ansi-language:EN-GB'>
tools to understand the assembly of FLU segmented genome <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Victor Hugo
Mello, Filipa <span class=SpellE>Sena</span>; Maria João Amorim<o:p></o:p></span></p>

<p class=MsoSubtitle><span lang=EN-GB style='mso-ansi-language:EN-GB'>Instituto
Gulbenkian de Ciência, Oeiras -Portugal<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-GB style='mso-ansi-language:EN-GB'>Influenza A
virus threatens human health, provoking seasonal epidemics and recurrent pandemic
outbreaks. Understanding viral dynamics is of utmost importance to avoid novel
pandemic outbreaks and mitigate the burden of seasonal flu. One critical aspect
for the formation of novel influenza viruses is the assembly of its segmented
genome, which involves the production of an eight-partite complex by an unknown
process that involves RNA-RNA interactions. Previously, we proposed that the
Influenza A Virus (IAV) genome assembles in viral-induced structures, namely
viral inclusions, that display the properties of liquids. However, the
molecular mechanism governing the phase separation of viral inclusions is
unclear. In my project, we explore the relationship between phase separation
and the assembly of the IAV genome. We hypothesize that the liquid properties
of these inclusions provide the environment required for genome assembly. To
investigate this hypothesis, we labelled a component of the IAV genomic complex
with a constitutively fluorescent protein and constructed fluorescent flu
viruses. This approach allows us to track the viral genome, measure its
dynamics and subcellular localization by live-cell imaging. Here, we present
the characteristics of our <span class=GramE>replication-competent</span>
fluorescent IAV synthesized with a reverse genetics approach. We built a
plasmid coding for the subunit of IAV polymerase PA (polymerase acidic), a
component of viral ribonucleoproteins (<span class=SpellE>vRNPs</span>), fused
with <span class=SpellE>mNeonGreen</span>. We reconstructed viral particles,
measured viral <span class=SpellE>titers</span> by plaque assay, and verified
the fluorescence signal after 16h of infection in lung cells. Viral growth in
multicycle infections did not result in a substantial decrease in viral <span
class=SpellE>titers</span> when compared to the wild <span class=GramE>type</span>
viral strain. Also, it was possible to identify clusters of PA-<span
class=SpellE>mNeonGreen</span> in liquid organelles, as previously reported for
IAV. These colocalized with both Rab11 and NP, a host factor that interacts
with the polymerase and another component of <span class=SpellE>vRNPs</span>,
respectively. Our results show that <span class=SpellE>mNeonGreen</span>-tagged
virions yield brighter emission signal when compared to the previously
described GFP-tagged ones. Altogether, these results validate our viral
construct as a helpful tool to investigate influenza genome assembly. <o:p></o:p></span></p>

<p class=Normal1><span lang=EN-GB style='mso-ansi-language:EN-GB'><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
